\chapter{An Application of risk prediction models to the Swedish Multi-Generational Breast Cancer Registry}{Joint work with Kamila Czene.\footnote{Department of Medical Epidemiology and Biostatistics, Karolinska Institutet, Solna, 171 77, Sweden, email: kamila.czene@ki.se.} In progress.}
\markboth{\textsc{Risk prediction models for breast cancer}}{\textsc{Risk prediction models for breast cancer}}\label{chapter:1}
% \author[1]{\fnm{Maria Veronica} \sur{Vinattieri}}\email{maria.vinattieri@phd.unibocconi.it}

% \author[2]{\fnm{Marco} \sur{Bonetti}}\email{marco.bonetti@unibocconi.it}
% % \equalcont{These authors contributed equally to this work.}

% \author[3]{\fnm{Kamila} \sur{Czene}}\email{kamila.czene@ki.se}
% % \equalcont{These authors contributed equally to this work.}

% \affil[1]{\orgdiv{Department of Decision Sciences}, \orgname{Bocconi University}, \orgaddress{\city{Milan}, \postcode{20136}, \state{Italy}}}

% \affil[2]{\orgdiv{Carlo F. Dondena Research Center, Bocconi Institute of Data Science and Analytics, Department of Social and Political Sciences}, \orgname{Bocconi University}, \orgaddress{\city{Milan}, \postcode{20136}, \state{Italy}}}

% \affil[3]{\orgdiv{Department of Medical Epidemiology and Biostatistics}, \orgname{Karolinska Institutet}, \orgaddress{\city{Solna}, \postcode{171 77}, \country{Sweden}}}

\begin{abstract}
We aim to study a specific aspect of the breast cancer development: the family-specific risk. We assume that the family-specific risk of developing breast cancer is latent and unchanged from birth. Our objective is to identify the highest-risk families, i.e. those with the highest values of family-specific risk, and target them towards more intensive screening and prevention strategies to enhance the probability of successful treatment. In contrast with the risk prediction models from the literature which include risk factors associated to breast cancer, we develop a parametric Multivariate frailty Cure-Rate survival model that takes into account the subjects as part of a family. This model incorporates a cure-rate component, allowing us to identify the fraction of the population not susceptible to breast cancer onset, the cured fraction, shedding light on the magnitude of the phenomenon of breast cancer development within the population. Simultaneously, our model maintains high accuracy in risk prediction on par of the semiparametric counterpart: the Multivariate frailty Cox model.

To assess the superior performance of our model, we conduct a comparative analysis with other models, including Cox models and models involving a strong risk factor associated to breast cancer: the family history indicator. Our comparison is illustrated using simulation studies and a real case dataset, namely the Multi-Generational Swedish Breast Cancer registry. Our assessment criteria include accuracy in risk prediction, measured by the area under the ROC curve (AUC) and the Harrell's Concordance index.

Our findings not only highlights the crucial role of incorporating complete family information in identifying high-risk families, in contrast to the limited utility of a family history indicator, but also demonstrates that the Multivariate frailty Cure-Rate model can elucidate the dynamics of breast cancer development within the population, by estimating the cured fraction and the distribution of breast cancer cases, without sacrificing accuracy in risk prediction, striking a balance between explanation and prediction.

\textbf{keywords}: breast cancer, family history, frailty models, risk prediction
\end{abstract}
%
\section{Introduction}\label{sec1}
Breast cancer risk prediction models have been extensively developed to tailor screening and prevention strategies. Several subject-specific risk factors associated to breast cancer have been involved to explain the variability of the breast cancer development and improve the predictive power of many predictive models. For example among three of the main references in the literature, there is Gail \cite{gail1989projecting} that uses the number of first-degree relatives who experienced breast cancer to predict the long-term probability of developing breast cancer. Rosner and Colditz that \cite{rosner2008risk} use, among the others, several reproductive covariates as the age at menarche, the age at menopause, and the age at childbirth to predict the incidence of breast cancer over a specified time period. And finally, Tyrer and Cuzick \cite{tyrer2004breast} have the same aim of predicting the subject-specific risk to develop breast cancer over a specific time period by accounting for risk factors, like reproductive covariates, family history, mammographic breast cancer density (MD) \cite{taroni2013optical}, body mass index (BMI), and life-style covariates. What we notice from the the aforementioned models is that they always model subject-specific time-to-event. The limitation here is that they include the hereditary component of breast cancer only through subject-specific covariates. For example, the number of first-degree breast cancer cases is a limited measure of the complete breast cancer history in a family, as it does not allow us to know the specific history of those cancers. 

% All of these models make use of subject-specific data, without account explicitly of the familiar pair-wise relationships between subjects that we retain an incredible additional source of information about breast cancer development. We aim to account for the familiar component and, both provide a better understanding of the phenomenon and increase the prediction accuracy of family-specific probability of occurring breast cancer diagnosis. 

Different from the others, the BOADICEA model \cite{antoniou2002comprehensive, antoniou2004boadicea, antoniou2008boadicea, thomas2004statistical} tried to directly incorporate the hereditary component of breast cancer, by inferring a  subject-specific genetic latent quantity, the polygenic component, that consists in the contribution of several genetics mutations with a singular small effect, on the disease development. They aim to estimate the familial risk of developing breast cancer for predicting cancer risk according to the family history of the disease and other risk factors. 
% The BOADICEA model is based on a subject-specific hazard, and it evaluates likelihood through a complex segregation analysis implemented in the Mendel software \cite{lee2019boadicea}. 
The BOADICEA is the most efficient tool for breast cancer risk prediction seen so far. Nevertheless, it is a complex tool that requires access to detailed subject-specific genetic and family history information. We aim to create a tool that is at the same time efficient but also easy to understand and use with accessible data.

Our novel contribution is found in retrieving the familiar relationship between subjects to jointly model subject-specific time-to-events within families. This allows us to know both the specific history of breast cancer cases and the family history. 

% In our dataset we simultaneously keep the subject-specific data and create a network structure all over the subjects to add the familial component in addressing the problem. Certainly, as subject-specific data are commonly available from a questionnaire or other similar and easily accessible sources, on the contrary a complete familiar network dataset is not easily available since the subject-specific dataset must be matched to a dataset containing the pair-wise relationships between subjects. To the best of our knowledge there is not a risk prediction model that combines subject-specific data with family network data as we do in this work. 

The available data allows us to jointly model the times-to-breast cancer onset under the conditional independence assumption given the family-specific frailty risk. We aim to ultimately infer the hereditary susceptibility to breast cancer by estimating the posterior distribution of the family-specific frailty risk and predicting the risk. This allows us to target high-risk families that are those with a higher value of the frailty risk. This aims to address the highest-risk families to specific screening and prevention strategies. Also, it is worth noting that we can infer the posterior distribution of the family-specific frailty risk also for women not included in the initial dataset. Therefore, it is crucial for new patients to provide familial breast cancer information to obtain a reliable estimate of the probability of belonging to one of the highest-risk families. 

We call our model the Multivariate frailty Cure-Rate model where ``multivariate'' refers to jointly modelling the times-to-event within families, in contrast to ``univariate'' where the subject-specific time-to-event is modelled; ``frailty'' refers to the frailty risk given its latent nature; and ``cure-rate'' to the peculiar structure that we give to the survival function as we assume that not all women will experience the breast cancer onset eventually.

A comparative analysis is conducted to evaluate the benefits of using the Multivariate frailty Cure-Rate model. We start from a model with includes a strong risk factor associated to breast cancer development: the first-degree family history indicator. This takes value one if at least one family member has experienced the breast cancer onset, or zero if none have, and it can be measured either at the end of the follow-up period or as time-varying during the follow-up. This model motivates the use of univariate models, such as also the Univariate frailty Cure-Rate model, that is the univariate counterpart of the Multivariate frailty Cure-Rate model. Lastly, also the known and widespread Cox model, in several of its forms, is included in the comparative analysis to assess that the predictive power of the Multivariate frailty Cure-Rate model reaches the same accuracy of the Multivariate frailty Cox model. Furthermore the Multivariate frailty Cure-Rate model, at contrary of the Multivariate frailty Cox model, reaches higher precision in inference and explanation of the phenomenon by estimating the cured fraction and the distribution of breast cancer cases. 

Section \ref{sec2} provides an overview of the background of the setting. In Section \ref{sec3} we describe the specific models employed in the analysis, and in Section \ref{sec4} we focus on the procedure of predicting the posterior risk. The simulation results are presented in Section \ref{sec5}, while the findings from the real case dataset, the Multi-Generational Swedish Breast Cancer registry, are discussed in Section \ref{sec:1.6}. 

\section{Background}\label{sec2}
We model the age (in years) $T$ of breast cancer onset, where the conditional hazard function is denoted by $\lambda(t\mid r)$, where $R = r$ indicates the frailty risk.  

The Univariate frailty model \cite{duchateau2008frailty} allows the hazard function to depend on the frailty quantity $R$ to capture the unobserved heterogeneity among subjects. The conditional hazard function at time $t$ is given by \[\lambda(t\mid r) = \alpha(r)\lambda_0(t),\] where $\alpha(r)$ can be any continuous function of the risk, and $\lambda_0(t)$ is the baseline hazard function corresponding to the value of the risk $r$ such that $\alpha(r)=1$. 

In presence of a vector of subject-specific covariates $x_j$ for the subject $j$, the frailty risk explains the unobserved heterogeneity that the covariates are not able to capture. The conditional hazard function at time $t$ for subject $j$, with frailty risk $r_j$, results \[\lambda_j(t\mid r_j) =\alpha(r_j)\lambda_0(t;x_j).\] 

In the Multivariate frailty model, the hazard function includes a frailty risk that is family-specific because subjects within the same family share the same frailty risk. The conditional hazard function at time $t$ for subject $j$ of family $i$ id given by \[ \lambda_j(t\mid r_i) = \alpha(r_i)\lambda_0(t;x_{ij}), \] with $x_{ij}$ the subject-specific vector of covariates and $r_i$ the risk of family $i$. 

One can choose several form of the function  $\alpha(r_i)$, as a polynomial form, an exponential form, or other (non-negative) forms. One may implement a hazard function for subject $j$ of family $i$ that is linearly dependent on the frailty, i.e.
\[
\lambda_j(t\mid r_i) = r_i\lambda_0(t; x_{ij})
\] \cite{duchateau2008frailty, ripatti2000estimation}. 
Notice that this readily coincides with the model specification, known also as Lehmann structure, given by \[\Lambda_j(t\mid r_i) = r_i\Lambda_0(t; x_{ij}), \qquad S_j(t\mid r_i) =  [S_0(t;x_{ij})]^{r_i}, \] where $\Lambda$ indicates the cumulative hazard function, and $S$ indicates the survival function. Notice that for proving the model identifiability, $\mathbb{E}(R) = 1$ is usually assumed and allows by the frailty risk distribution. 

In the Multivariate frailty model, the family-specific joint survival function is computed under the conditional independence assumption \cite{rodriguez2010multivariate}. Consider the case with only two women belonging to the same family. Let $T_1 = t_1$ and $T_2 = t_2$ be the times-to-event of the two women, with survival function $S_1$ and $S_2$ from family $i$, with $r_i$ be their  family-specific frailty risk. By the conditional independence assumption one has that the joint survival function factorizes conditional to the frailty risk: \[
S_{12}(t_1,t_2\mid r_i) \overset{T_1\perp T_2\mid R}{=} S_1(t_1\mid r_i)S_2(t_2\mid r_i).
\] 

And the unconditional joint survival function is equal to \begin{align*}
    S_{12}(t_1,t_2) = \int_{0}^{\infty} S_{12}(t_1,t_2\mid r)g(r)dr &= \int_{0}^{\infty} e^{-r(\Lambda_{01}(t_1)\Lambda_{02}(t_2))} g(r)dr \\
    &=\mathcal{L}_g(\Lambda_{01}(t_1)\Lambda_{02}(t_2))
\end{align*} where $\mathcal{L}_g(r)$ is the Laplace transform of the density function $g(r)$ of $R$ evaluated at $\Lambda_{01}(t_1)\Lambda_{02}(t_2)$. To obtain the explicit form of $S_{12}(t_1,t_2)$ one should specify $g(r)$, that is typically a Gamma or Lognormal distribution. This framework can be easily extended to more than two subjects \cite{hougaard2012analysis}.

% MB: explain how one can use a non-parametric frailty here. 
% MB: explain where the Cox models enter here.

% it counts for the correlation among survival times in a family but not for the covariate effect. A more general random effects model is the proportional hazard mixed-effects model \begin{align*}
%     &\lambda_{ij}(t) = \lambda_0(t)\text{exp}(\beta'x_{ij} + b'_i W_{ij}) \\
%     &b_i\overset{i.i.d.}{\sim} N(0, \sigma^2)
% \end{align*} where the $W_{ij}$ are covariates with random effects. The inference is made through the full likelihood! 

% While an other way to model a common effect given by clustered data is through the marginal approach (WLW) \begin{align*}
%     &\lambda_{ij}(t) = \lambda_0(t)\text{exp}(\beta_j'x_{ij}) \\
%     &\beta_j \text{ is the failure-specific effect}
% \end{align*} for each subject $i$ for failure type $j$. In our case we can consider $x_{ij}$ different treatments/groups, then high/low risk group. 

% The other aspect to consider is the multivariate setting of the problem. We rely on Multivariate Survival function \cite{hougaard2012analysis}. The name ``multivariate'' holds for referring to many outcomes (time-to-event), not just many explanatory variables \cite{rodriguez2010multivariate}. Taking into account the problem of family history in breast cancer, if two women have their time-to-event similar within the same family, then their time-to-event are dependent: we are facing a case of clustered survival data where the cluster is the family itself. Conditionally to the family their time-to-event are completely independent. Let's look at this issue in the scenario of the bivariate frailty models, denoting with $\theta$ the family membership parameter, where it represents a random effect (genetic or environmental effects) such that conditionally to it, the time-to-event, let's say $T_1$, $T_2$ are independent. This case can be readily generalized to more than two survival times. The assumption of \textbf{Conditional Independence} can be written as follows \begin{align*}
%     S_{12}(t_1,t_2\mid \theta) = S_1(t_1\mid \theta)S_2(t_2\mid \theta).
% \end{align*}

\section{Model Specification}\label{sec3}
Let $i=1,\dots,n$ identify the family and the main subject of a family, and $\{m,s_1\}$ to identify the two family members ``mother'' and ``first sister'', respectively. The generalization of this to families with more than three members is straightforward. The observed survival data is $\underline X=\left(\underline X_1,\dots,\underline X_n\right)^T$, where $\underline X_i=(\underline x_i, \underline{xs}_{1i}, \underline xm_i)^T$. 
For the generic subject $i$, $\underline x_i=(x_{i}=\text{min}(t_{i}, c_{i}),\delta_{i})^T$, $\delta_{i}=\mathbb{I}(t_{i} \le c_{i})$ following the usual notation where $t_i$ indicates the survival time, and $c_i$ indicates the administrative (independent) censoring time, both measured from the same origin, that in our case is the birth $b_i$. The notation for the other family members is obtained by having $x, \ t, \ c, \ b$, and $\delta$ be followed by $m$, and  $s_1$. If subject $i$ does not have a sister, then the sister birthday is set as $bs_{1i} = +\infty$ and consequently the time-to-event is $ts_{1i} = +\infty$. The distinction between mother and sister is not strictly needed here as we assume that their time-to-event distributions are equal but it will make the extension to more complex models easier.

We extend the Lehmann structure in terms of the survival functions $S_r(t)=[S_0(t)]^r$ to the Cure-Rate model. We call this model the  Lehmann Cure-Rate (LCR) model obtained by applying the Lehmann power transformation to a Cure-Rate (CR) survival function \[CR: \ S_0(t) = p + (1-p) \widetilde{S}(t)\] 
\[LCR: \ S_r(t) = \left[ p + (1-p) \widetilde{S}(t) \right]^r, \ \ r >0,  \] with $p$ the cured fraction, and $\widetilde{S}(t)$ a proper survival function which describes the time-to-event distribution of the cases, which are the subjects who will eventually experience the event, with some parameters $\underline\gamma$. Thus, the time-to-event of cases do not admit the value $+\infty$, and their survival function is proper, i.e. $\lim_{t\to+\infty}\widetilde S(t) = 0$. Note that for a fixed value $r$, the survival function $S_r(t)$ also defines a Cure-Rate model. Indeed, $\lim_{t \rightarrow +\infty} S_r(t) = p^r$, and $S_r(t)$ can thus be written as
\begin{eqnarray}
S_r(t) = p^r + \left( 1-p^r \right) \widetilde{S}_r(t),
\label{CR}
\end{eqnarray}
with proper conditional survival function for the cases equal to
\begin{eqnarray}
\widetilde{S}_r(t) = \dfrac{\left[ p + (1-p) \widetilde{S}(t) \right] ^r - p^r}{1-p^r},
\label{condsurv}
\end{eqnarray}
and proper conditional density function for the cases equal to
\begin{eqnarray}
\widetilde{f}_r (t) = - \dfrac{d}{d t} \widetilde{S}_r(t) = \dfrac{1-p}{1-p^r} r \left[ p + (1-p) \widetilde{S}(t) \right]^{\left(r -1 \right)} \widetilde{f}(t).  
\label{formula:1.conddens}
\end{eqnarray}

Recall that the cure-rate models as in \ref{CR} refers to survival random variable $T$ with improper cumulative distribution function. Indeed, we have that $P(T=+\infty)=p>0$, and also a proper density function does not exist. If $S_0(t) = p + (1-p)\widetilde S(t)$ follows a cure-rate structure, we have
\begin{align}
    \label{formula:lemma_1}
    \lim_{t\to+\infty}S_0(t)= \lim_{t\to+\infty}(p+(1-p)\widetilde{S}(t))=p>0.
\end{align}
Suppose that a proper density function $f(t)$ exists, so that $S_0(t)=\int_t^\infty f(u)\text{d}u$ and $\int_0^\infty f(u)\text{d}u=1$. 
\begin{align*}
    &\text{Since }\mathbb{I}(s\ge t)f(s) \le f(s) \ \forall s\ge 0 \text{ and } \int_0^{+\infty} f(s)\text{d}s=1, \text{ one has} \\
    &0 \le \int_0^{+\infty}\mathbb{I}(s\ge t)f(s) \le \int_0^{+\infty} f(s)\text{d}s = 1. \text{ Then by the DCT \cite{weir1973lebesgue},}\\
    &\lim_{t\to+\infty}S_0(t) = \lim_{t\to+\infty}{\int_{t}^{+\infty} f(s)\text{d}s} = \lim_{t\to+\infty}{\int_{0}^{+\infty}\mathbb{I}(s\ge t)f(s)\text{d}s}\overset{\overset{DCT}{\downarrow}}{=} \int_{0}^{+\infty}\lim_{t\to+\infty} [\mathbb{I}(s\ge t)f(s)]\text{d}s=0.
\end{align*}
Hence $\lim_{t \to+\infty}S_0(t) = 0$, which contradicts \ref{formula:lemma_1}. Additional comments on the cure-rate structure and PH assumption can be found in Appendix \ref{appendix:1.a}. 

We now specify the models and their likelihood that we will use later for the comparative analysis. 

\subsection{Models}\label{sec:1.3.1}
In the literature we can find the use of the family history indicator as a strong risk factor associated to breast cancer. We use the family history as an indicator of the risk of developing breast cancer within a family. The family history indicator  $FH$ has value zero until the first case of breast cancer is observed in the family, after which it takes the value one.  We thus develop a Univariate $FH$ Cure-Rate model which includes the covariate $FH$. Notice that the survival function depends on the baseline covariate $FH$, and it is given by  \begin{align*}
    &S_{FH}(t) = S_0(t)^{(1-fh)}S_0(t)^{fh\beta} = S_0(t)^{fh(\beta-1)+1} = [p+(1-p)\widetilde S(t)]^{fh(\beta-1)+1} \\
    &\text{or, } S_{FH}(t) = p^{fh(\beta-1)+1} + (1-p^{fh(\beta-1)+1})\widetilde S_{FH}(t) \\ 
    &\text{with }\widetilde S_{FH}(t) = \dfrac{[p+(1-p)\widetilde S(t)]^{fh(\beta-1)+1} - p^{fh(\beta-1)+1}}{(1-p^{fh(\beta-1)+1})}, \\
    &\text{and } f_{FH}(t) = (1-p^{fh(\beta-1)+1})\widetilde f_{FH}(t)= (fh(\beta-1)+1)(1-p)\widetilde f(t) [p+(1-p)\widetilde S(t)]^{fh(\beta-1)}, \\
    &\text{with } \widetilde f_{FH}(t) = -\dfrac{\partial \widetilde S_{FH}(t)}{\partial t} = \dfrac{(fh(\beta-1)+1)[p+(1-p)\widetilde S(t)]^{fh(\beta-1)} (1-p)\widetilde f(t)}{(1-p^{fh(\beta-1)+1})}, 
\end{align*} for $FH = fh$, and with $\widetilde S_{FH}(t)$ the survival function of cases . The parameter $\beta$ identifies the average risk difference between the group of families with a negative family history ($FH = 0$), and those families with a positive family history ($FH = 1$).  Notice that when $FH = 0$ the survival function reduces to the baseline survival function $S_{FH}(t;FH=0) = [p+(1-p)\widetilde S(t)]$, otherwise $S_{FH}(t;FH=1) = [p+(1-p)\widetilde S(t)]^\beta$, which allows for a cure-rate model as the previous case. 

The Univariate FH Cure-Rate model has a univariate likelihood on the parameter collection $\pi_{FH}=(\underline\gamma, p, \beta)^T$, that is given by \begin{align*}
    L(\pi_{FH})&=\prod_{i=1}^n f_{FH}(x_i)^{\delta_i} S_{FH}(x_i)^{1-\delta_i} \\
    &=\prod_{i=1}^n\left[(fh_i(\beta-1)+1)(1-p)\widetilde f(x_i)\right]^{\delta_i} \left[p+(1-p)\widetilde S(x_i)\right]^{fh_i(\beta-1)}.
\end{align*}

This model motivates the use of a binary risk of developing breast cancer, but we believe that a binary risk is too simplistic. For this reason we move to the development of models with a continuous frailty risk. Moreover, the frailty risk must be positive. We assume the frailty to follow a Gamma($shape = \theta$, $rate = \theta$) whose density function is given by \begin{align*}
    g_R(r) = \dfrac{\theta^\theta}{\Gamma(\theta)}r^{\theta-1}\text{e}^{-r\theta},
\end{align*}
where we $\theta$ the frailty parameter. An extension to other distributions, or with a number of parameters greater than one is straightforward.

We develop two models with the continuous frailty risk: the Multivariate frailty Cure-Rate model and the Univariate frailty Cure-Rate model. 

We can find the closed form of the multivariate likelihood $L(\underline\gamma, p, \theta)$ of the Multivariate frailty Cure-Rate model for $i = 1,\dots, n$ families of varying size $n_i$ by applying the following steps:
\begin{align*}
    &L(\underline\gamma, p, \theta) = \prod_{i=1}^n\prod_{j=1}^{n_i}\int_{\mathbb {R}^+}f_r(x_{ij})^{\delta_{ij}}S_r(x_{ij})^{1 - \delta_{ij}}g_R(r;\theta)\text{d}r \\
    &=\prod_{i=1}^n\int_{\mathbb{R}^+}\prod_{j=1}^{n_i}\left[\dfrac{(1-p)\widetilde f(x_{ij})}{p + (1-p)\widetilde S(x_{ij})}r\cancel{(p + (1-p)\widetilde S(x_{ij}))^r}\right]^{\delta_{ij}}[p + (1-p)\widetilde S(x_{ij})]^{r(1-\cancel{\delta_{ij}})}g_R(r;\theta)\text{d}r \\
    &=\prod_{i=1}^n\prod_{j=1}^{n_i}\left[\dfrac{(1-p)\widetilde f(x_{ij})}{p + (1-p)\widetilde S(x_{ij})}\right]^{\delta_{ij}}\int_{\mathbb R^+}\prod_{j=1}^{n_i} r^{\delta_{ij}}S_r(x_{ij})g_R(r;\theta)\text{d}r \\
    &=\prod_{i=1}^n\prod_{j=1}^{n_i}\left[\dfrac{(1-p)\widetilde f(x_{ij})}{p + (1-p)\widetilde S(x_{ij})}\right]^{\delta_{ij}}\int_{\mathbb R^+}r^{\sum_{j=1}^{n_i}\delta_{ij}}\prod_{j=1}^{n_i}S_r(x_{ij})g_R(r;\theta)\text{d}r
\end{align*} 
Thus, given the general distribution $R\sim$Gamma$(shape = \alpha, rate = \beta)$, the development of the internal component is given by 
\begin{align*}
    &\int_{\mathbb R^+}r^{\sum_{j=1}^{n_i}\delta_{ij}}\prod_{j=1}^{n_i}S_r(x_{ij})g_R(r;\theta)\text{d}r = \int_{\mathbb R^+}\prod_{j=1}^{n_i}S_r(x_{ij})r^{\sum_{j=1}^{n_i}\delta_{ij}}\dfrac{\beta^\alpha}{\Gamma(\alpha)}r^{\alpha-1}\text{e}^{-\beta r}\text{d}r \\ 
    &= \int_{\mathbb R^+}\prod_{j=1}^{n_i}S_r(x_{ij})\dfrac{\beta^{\alpha + \sum_{j=1}^{n_i}\delta_{ij}}}{\Gamma(\alpha + \sum_{j=1}^{n_i}\delta_{ij})}\dfrac{\Gamma(\alpha + \sum_{j=1}^{n_i}\delta_{ij})}{\Gamma(\alpha)\beta^{\sum_{j=1}^{n_i}\delta_{ij}}}r^{(\alpha+\sum_{j=1}^{n_i}\delta_{ij})-1}\text{e}^{-\beta r}\text{d}r \\ 
    &= \int_{\mathbb R^+}\prod_{j=1}^{n_i}S_r(x_{ij})\dfrac{\beta^{\alpha + \sum_{j=1}^{n_i}\delta_{ij}}}{\Gamma(\alpha + \sum_{j=1}^{n_i}\delta_{ij})}\dfrac{\Gamma(\alpha + \sum_{j=1}^{n_i}\delta_{ij})}{\Gamma(\alpha)\beta^{\sum_{j=1}^{n_i}\delta_{ij}}}r^{(\alpha+\sum_{j=1}^{n_i}\delta_{ij})-1}\text{e}^{-\beta r}\text{d}r \\ 
    &=\prod_{j=1}^{n_i}\dfrac{\Gamma(\alpha + \sum_{j=1}^{n_i}\delta_{ij})}{\Gamma(\alpha)\beta^{\sum_{j=1}^{n_i}\delta_{ij}}}\int_{\mathbb R^+}H(x_{ij};p,\underline\gamma)^rg_{R^*}(r;\alpha,\sum_{j=1}^{n_i}\delta_{ij},\beta)\text{d}r \\ 
    &=\prod_{j=1}^{n_i}\dfrac{\Gamma(\alpha + \sum_{j=1}^{n_i}\delta_{ij})}{\Gamma(\alpha)\beta^{\sum_{j=1}^{n_i}\delta_{ij}}} \int_{\mathbb R^+}
    \text{e}^{r\text{log}(H(x_{ij};p,\underline\gamma))}g_{R^*}(r;\alpha,\sum_{j=1}^{n_i}\delta_{ij},\beta)\text{d}r \\ 
    &=\prod_{j=1}^{n_i}\dfrac{\Gamma(\alpha + \sum_{j=1}^{n_i}\delta_{ij})}{\Gamma(\alpha)\beta^{\sum_{j=1}^{n_i}\delta_{ij}}} \mathbb{E}_{R^*}[\text{e}^{r\text{log}(H(x_{ij};p,\underline\gamma))}] =\prod_{j=1}^{n_i}\dfrac{\Gamma(\alpha + \sum_{j=1}^{n_i}\delta_{ij})}{\Gamma(\alpha)\beta^{\sum_{j=1}^{n_i}\delta_{ij}}}\text{MGF}(R^*;\text{log}(H(x_{ij};p,\underline\gamma))) \\
    &=\prod_{j=1}^{n_i}\dfrac{\Gamma(\alpha + \sum_{j=1}^{n_i}\delta_{ij})}{\Gamma(\alpha)\beta^{\sum_{j=1}^{n_i}\delta_{ij}}}\left(1 - \dfrac{\text{log}(H(x_{ij};p,\underline\gamma))}{\beta}\right)^{-(\alpha + \sum_{j=1}^{n_i}\delta_{ij})}
\end{align*}
where we define the quantity $H(x_{ij};p,\underline\gamma) = \prod_{j = 1}^{n_i}S(x_{ij}) = \prod_{j = 1}^{n_i}\left(p + (1-p)\widetilde S(x_{ij})\right)$ for ease of writing. Later we update the parameters of the frailty distribution to obtain the updated frailty risk random variable $R^*\sim$Gamma$(shape = \alpha+\sum_{j=1}^{n_i}\delta_{ij},rate=\beta)$, so that we make use of its moment generating function (MGF) in the point $\log(H(x_{ij};p,\underline\gamma))$. Recall that the definition of the MGF implies that \[
MGF(R;y) = \mathbb{E}_R[\text{e}^{ry}],
\] which is exactly what we have. Specifically, for a Gamma distributed random variable $R\sim\text{Gamma}(shape = \alpha, rate = \beta)$ the MGF is given by \[
MGF(R;y) = \left(1-\dfrac{y}{\beta}\right)^{-\alpha}.
\]
Thus, the multivariate likelihood with $\alpha=\beta=\theta$ is given by \begin{align*}
    &L(\underline\gamma, p, \theta)=\prod_{i=1}^n\prod_{j=1}^{n_i}\left[\dfrac{(1-p)\widetilde f(x_{ij})}{p + (1-p)\widetilde S(x_{ij})}\right]^{\delta_{ij}}\dfrac{\Gamma(\theta + \sum_{j=1}^{n_i}\delta_{ij})}{\Gamma(\theta)\theta^{\sum_{j=1}^{n_i}\delta_{ij}}}\left(1 - \dfrac{\log(H(x_{ij};p,\underline\gamma))}{\theta}\right)^{-(\theta + \sum_{j=1}^{n_i}\delta_{ij})}.
\end{align*} 
The likelihood, always in the case $\alpha=\beta=\theta$, reduces to the univariate form when there is one subject per family, so that its form is given by \begin{align*}
    &L(\underline\gamma, p, \theta)=\prod_{i=1}^n\left[\dfrac{(1-p)\widetilde f(x_i)}{p + (1-p)\widetilde S(x_i)}\right]^{\delta_{i}}\dfrac{\Gamma(\theta + \delta_{i})}{\Gamma(\theta)\theta^{\delta_{i}}}\left(1 - \dfrac{\text{log}(p + (1-p)\widetilde S(x_i))}{\theta}\right)^{-(\theta + \delta_i)}.
\end{align*}
Notice that, at this point, parameter estimation can be achieved through the maximization of the likelihood from the observed independently right-censored data. Once the estimated collection of parameters $\underline{\widehat\pi} = (\widehat p, \underline{\widehat\gamma}, \widehat\theta)^T$ is obtained, it can be consequently involved in the computation of quantities of interest.

If one is willing to ignore the Cure-Rate structure, one may think of a model that has a proper survival distribution, both conditionally on $R$ and marginally. In other words, we may assume that there exists some finite time $T_\text{max}$ (larger than all observed survival and censoring times) such that all conditional survival functions $S(t\mid r) = \left[S_0(t) \right]^r$ are proper, i.e. they tend to zero as $t\to+\infty$. 

The popular and widely developed Cox model may be suitable for our case. We include in the comparative analysis a Multivariate frailty Cox model, where the semiparametric estimation of the model is possible, for example using the \texttt{emfrail} function available in the software \texttt{R} \cite{balan2019frailtyem, balan2020tutorial}, while maintaining the multiplicative frailty structure. This Multivariate frailty Cox model is the exact counterpart of the Multivariate frailty Cure-Rate model, which we would like to prove they have an equal level of prediction accuracy. For sake of completeness of the analysis, we also develop a Univariate $FH$ Cox model, and a Univariate $FH(t)$ Cox model, with $FH(t)$ the time-varying version of the family history indicator.  The difference between these two is that the indicator $FH$ is measured at the end of the follow-up, while $FH(t)$ is measured over time and the time of the change-point from zero to one is tracked. Unfortunately it was not possible to include in the analysis the Univariate frailty Cox model because of its non-identifiability due to the combination of the absence of families and an unspecified hazard function. We prove the non-identifiability of the univariate Cox frailty model in the following lines.

Let us start from the multiplicative proportional hazards frailty model that has the observed data likelihood given by
\begin{align*}
    \label{}
    &L^*(\underline{\pi};\underline x) =\prod_{i=1}^n f_X(\underline{x}_i;\underline{\pi}) = \prod_{i=1}^n\int_{\mathbb{R}^+}f_X(\underline{x}_i\mid R;\underline{\pi})g_R(r)\text{d} r=\prod_{i=1}^n\int_{\mathbb{R}^+}f_r(x_i)^{\delta_i}S_r(x_i)^{1-\delta_i}g_R(r)\text{d} r,   
\end{align*} where recall $\underline\pi = (p,\underline\gamma,\theta)^T$ and $S_r(t) = S(t\mid r)$ and $f_r(t) = f(t\mid r)$ are the (proper) survival function and density function conditional on the cases, respectively.

We note that, however, the Univariate frailty model is not identifiable without some additional structure. We dig deeper into the issue with the following lemma.
\begin{lem} The Univariate frailty model is not identifiable if one does not specify the form of the baseline survival function $S_0(t)$. 
    \begin{proof}
Recall the Gamma density function:
\[
g_R(r;\alpha,\beta) = \dfrac{1}{\Gamma(\alpha)}\beta^\alpha r^{\alpha - 1}\text{e}^{-r\beta}, \ r\ge 0, \ \alpha, \ \beta\in\mathbb{R}^+
\]
Letting $\alpha=\beta=\theta > 0$ yields \[
g_R(r;\theta) = \dfrac{1}{\Gamma(\theta)}\theta^{\theta}r^{\theta-1}\text{e}^{-r\theta}
\]
Recall the moment generator function of the Gamma random variable: \begin{align*}
    &MGF_R(y) =\mathbb{E}[\text{e}^{yR}] = \left(1 - \dfrac{y}{\beta}\right)^{-\alpha}, \ y<\beta \ \text{or} \\
    &\left(1 - \dfrac{y}{\theta}\right)^{-\theta}\text{ when } \alpha=\beta=\theta>0, \text{ for } y < \theta, \text{ and in particular, } \forall \ y \le 0.
\end{align*} 
The Univariate frailty model has marginal survival function \begin{align*}
    S(t) &= \int_{\mathbb{R}^+}S(t\mid r)g_R(r;\theta)\text{d}r = \int_{\mathbb{R}^+}S(t)^rg_R(r;\theta)\text{d}r = \int_{\mathbb{R}^+}\text{e}^{\text{log}\left(S(t)^r\right)}g_R(r;\underline{\theta})\text{d}r \\
    &= \int_{\mathbb{R}^+}\text{e}^{r\text{log}\left(S(t)\right)}g_R(r;\theta)\text{d}r= \left(1 - \dfrac{\text{log}(S_0(t))}{\theta}\right)^{-\theta}. \text{ Note that, }\log(S_0(t)) < 0 \forall t\ge 0.
\end{align*}
Now, for any observed marginal survival function $S(t)$ one has \begin{align*}
    &S(t) = \int_{\mathbb{R}^+}S_0(t)^rg_R(r;\theta)\text{d}r = \left(1 - \dfrac{\log(S_0(t))}{\theta}\right)^{-\theta} \iff S(t)^{-1/\theta} = 1 - \dfrac{\log(S_0(t))}{\theta} \\
    &\iff S(t)^{-1/\theta} - 1 = -\dfrac{\log(S_0(t))}{\theta} \iff \theta\left[S(t)^{-1/\theta} - 1\right] = -\log(S_0(t)) \\
    &\iff \log(S_0(t)) = \theta\left(1 - S(t)^{-1/\theta} \right) \iff S_0(t) = \text{exp}\left(\theta\left(1 - S(t)^{-1/\theta} \right)\right),
\end{align*} 
for any observed $S(t)$, to each choice of $\theta$ there is a corresponding $S_0(t)$. Thus, it exists an infinite set of couple $(\theta, S_0(t))$ with $S_0(t) = \text{exp}\left(\theta\left(1 - S(t)^{-1/\theta} \right)\right)$ that yields to the same $S(t)$. This proves that the Univariate frailty model is not identifiable from the marginal survival function $S(t)$, unless additional constraints are introduced, such as for example on the distribution of $S_0(t)$.
    \end{proof}
\end{lem}
Thus, the Univariate frailty Cox model is not developed in the end and no results are reported from this model. 

Wrapping up, we explore six different models: the Multivariate frailty Cure-Rate model, the Univariate frailty Cure-Rate model, the Univariate $FH$ Cure-Rate model, the Multivariate frailty Cox model, the Univariate $FH$ Cox model, the Univariate $FH(t)$ Cox model. Our purpose is to observe their difference in inference precision and risk prediction accuracy. We expect the family history indicator to be a weaker indicator compared to the continuous frailty risk, in terms of explaining the phenomenon of breast cancer within a population. The Cox models can not infer some quantities as the cured fraction and the distribution of breast cancer cases, and this is certainly a disadvantage  for these models. Also, the univariate models do not use the complete familiar information of the subjects and for this reason we expect them to be worst than the multivariate models in terms of risk prediction accuracy. Summing up all of these thoughts, we expect the Multivariate frailty Cure-Rate model to be the best performing in inference precision to explain the phenomenon breast cancer development, and accuracy in risk prediction.

\section{Posterior risk prediction}\label{sec4}
The primary focus of this project is risk prediction. It is interesting to notice that the available data are used for fitting the models, the risk prediction can be applied also to women not included in the initial dataset. The risk prediction procedure is different according to whether the model is semiparametric or parametric. For this reason we make distinction in the following paragraphs. 

\subsection{Semiparametric scenario}
In the parameter estimation step, by maximizing the partial likelihood we obtain the estimated value of the frailty parameter $\widehat{\theta}$. 
% Through the already available \texttt{coxph} function in the \texttt{R} software, using an Empirical Bayes method implemented in the package to compute the posterior mean \cite{balan2020tutorial}, each family can be assigned to a risk value.

With the value of the frailty parameter we can obtain the estimated family-specific posterior frailty risk density. This step is important to quickly predict risk values without estimating parameters when a new family arises. The posterior distribution, for a family of three component, is given by \[
    \label{famspec-post-risk}
    g(r \mid \text{family data}; \widehat{\theta}) = 
    \dfrac{f(\text{family data}\mid R = r)g(r; \widehat{\theta})}{\int_{\mathbb{R}^+} f(\text{family data}\mid R=r)g(r; \widehat{\theta}) \text{d} r},
    \] 
where \[
f(\text{family data}\mid R=r) \overset{\overset{\perp\mid R}{\downarrow}}{=} f(\underline{x}\mid r)f(\underline{xm}\mid r)f(\underline{xs_1}\mid r), 
\] 
and \[f(\underline{x}\mid R = r) = f_r(x)^\delta S_r(x)^{1-\delta} = \left(\dfrac{f_r(x)}{S_r(x)}\right)^\delta S_r(x) = (r\lambda_0(x))^\delta S_0(x)^r.
\] 

A similar procedure is applied for the other family members, and also again, this is easily extendable to a higher number of female family members. At this point, an estimator for the baseline survival function $\widehat S_0(t)$ and for the baseline hazard function $\widehat\lambda_0(t)$ is needed such that one can estimate the subject-specific density function
\[
\widehat{f}(\underline{x}\mid R = r) = (r\widehat{\lambda}_0(x))^\delta\widehat{S}_0(x)^r.
\]

The baseline cumulative hazard function, and consequently the survival function $\widehat S_0(t) = \text{e}^{-\widehat\Lambda_0(t)}$, can be estimated by the Breslow estimator $\widehat\Lambda_0(t)$. 

In the following lines, we present the classical estimation of the Breslow estimator with covariates. We consider the expected number of events occurring at time $[\tau_j,\tau_{j+1}]$, given the $l$th subject ``at risk'' at time $\tau^-_j$, who belongs the risk group defined as $R(\tau_j)$. Thus, we have \begin{align*}
    &\underset{l\in R(\tau_j)}{\sum}(\tau_{j+1}-\tau_j)\lambda(\tau_j\mid \underline{x}_l) = \underset{l\in R(\tau_j)}{\sum}(\tau_{j+1}-\tau_j)e^{\underline{\beta}^{'}\underline{x}_l}\lambda_0(\tau_j).
\end{align*} 

When setting the aforementioned quantity equal to the observed number of events $d_j$ until time $\tau_{j+1}$, it yields \begin{align*}
    &d_j = \lambda_0(\tau_j)(\tau_{j+1}-\tau_j)\underset{l\in R(\tau_j)}{\sum}e^{\underline{\beta}^{'}\underline{x}_l} \\
    &\lambda_0(\tau_j)(\tau_{j+1}-\tau_j)\approx\int_{t_j}^{t_{j+1}}\lambda_0(u)\text{d}u = \dfrac{d_j}{\underset{l\in R(\tau_j)}{\sum}e^{\underline{\widehat{\beta}}^{'}\underline{x}_l}} \iff\widehat{\Lambda}_0(t)=\underset{\tau_j<t}{\sum}\left[\dfrac{d_j}{\underset{l\in R(\tau_j)}{\sum}e^{\underline{\widehat{\beta}}^{'}\underline{x}_l}}\right]
\end{align*} where $\underline{x}_l$ is the covariate vector for subject $l$, $\underline\beta$ is the parameter collection, and $\widehat{\Lambda}_0(t)$ is the Breslow estimator. 

The estimated value of the survival function, through Breslow estimator, is obtained through the available function \texttt{predictCox} in the package \texttt{riskRegression} \cite{ozenne2017riskregression} of the software \texttt{R}, which takes as input a Cox model, while the estimated hazard function can be obtained in the function \texttt{coxphHaz} of package \texttt{biostat3} \cite{biostat3} of the software \texttt{R}.

We need also to estimate the other component of formula \ref{famspec-post-risk}: the posterior distribution of the frailty risk $g(r;\widehat\theta)$. The computation of $g(r;\widehat\theta)$ is solved by relying on the conjugacy property of the Gamma prior for this model. Specifically, the posterior frailty risk distribution for a particular family is easily shown to be distributed as a Gamma distribution with shape parameter equal to $\theta + d_i$, with $d_i$ the number of observed events in family $i$, and rate parameter equal to $\theta + \sum_{j=1}^{n_i}\Lambda_0(x_j)$, where $\Lambda_0(\underline x_i) = \sum_{j=1}^{n_i}\Lambda_0(x_j)$, with $\underline x_i = (x_1, \dots, x_{n_i})^T$ for family size $n_i$, is the cumulative hazard function at time 0 evaluated at the observed times of the family's members. For a family of three members the posterior frailty risk distribution is given by \begin{align*}
    &g_R(r\mid \underline X) =  (r\widehat{\lambda}_0(x_i))^{\delta_i}\widehat{S}_0(x_i)^r(r\widehat{\lambda}_0(xm_i))^{\delta m_i}\widehat{S}_0(xm_i)^r(r\widehat{\lambda}_0(xs_{1i}))^{\delta s_{1i}}\widehat{S}_0(xs_{1i})^r g_R(r;\widehat{\theta}) \\
    &g_R(r\mid \underline X) = r^{\delta_i+\delta m_i+\delta s_{1i}}\left(\widehat{S}_0(x_i)\widehat{S}_0(xm_i)\widehat{S}_0(xs_{1i})\right)^r g_R(r;\widehat{\theta})\\
    &g_R(r\mid \underline X) \propto r^{\delta_i+\delta m_i+\delta s_{1i}}\text{e}^{r\left(-\widehat{\Lambda}_0(x_i)-\widehat{\Lambda}_0(xm_i)-\widehat{\Lambda}_0(xs_{1i})\right)}g_R(r;\widehat{\theta}) \iff
    % &g_R(r\mid \underline X) \propto r^{d_i}\text{e}^{r\left(-\widehat{\Lambda}_0(\underline X_i)-\widehat{\Lambda}_0(\underline Xm_i)-\widehat{\Lambda}_0(zs_{1i})\right)}r^{\theta-1}\text{e}^{-r\theta} \\
    g_R(r\mid \underline X) \propto r^{d_i+\theta-1}\text{e}^{-r\left(\theta + \sum_{j=1}^{n_i}\widehat{\Lambda}_0(x_j)\right)} \\
    &g_R(r\mid \underline X) = \dfrac{r^{d_i+\theta-1}\text{e}^{-r\left(\theta + \sum_{j=1}^{n_i}\widehat{\Lambda}_0(x_j)\right)}}{\int_{\mathbb{R}^+}r^{d_i+\theta-1}\text{e}^{-r\left(\theta + \sum_{j=1}^{n_i}\widehat{\Lambda}_0(x_j)\right)}\text{d} r} \\
    &g_R(r\mid \underline X) = \dfrac{r^{h_i(\theta)-1}\text{e}^{-r l_i(\theta) }}{\left(\dfrac{1}{l_i(\theta)}\right)^{h_i(\theta)}\Gamma(h_i(\theta))\int_{\mathbb{R}^+}\dfrac{1}{\left(\dfrac{1}{l_i(\theta)}\right)^{h_i(\theta)}\Gamma(h_i(\theta))}r^{h_i(\theta)-1}\text{e}^{-r l_i(\theta)}\text{d} r} \\
    &g_R(r\mid \underline X) = \dfrac{r^{h_i(\theta)-1}\text{e}^{-r l_i(\theta)}l_i(\theta)^{h_i(\theta)}}{\Gamma(h_i(\theta))},
\end{align*}
where $h_i(\theta) = d_i+\theta, \ l_i(\theta) = \theta + \sum_{j=1}^{n_i}\widehat{\Lambda}_0(x_j)$. The final formula is given by \begin{align*}
    g_R(r_i\mid \underline X) = \frac{r_i^{d_i+\theta-1}\text{exp}(-r_i(\theta+\widehat{\Lambda}_0(\underline x_i)))(\theta+\widehat{\Lambda}_0(\underline x_i))^{d_i+\theta}}{\Gamma(d_i+\theta)}.
\end{align*} 

This formula is also computed with a different parametrization of the Gamma distribution \cite{duchateau2008frailty}, for the maximization step of the EM algorithm in order to achieve parameter estimation. 

Notice that to obtain the posterior frailty distribution of each family we need to evaluate the cumulative hazard function at the observed times within each family. This means that, if the estimated cumulative hazard value is not available for that observed time, we use linear interpolation on the available data points \cite{meijering2002chronology}.

Typically, the raw posterior frailty risk distribution has a poor meaning to the end-user. We need to communicate to each family a meaningful summary of it as, for example, the mode, median or mean, whose equations are given by \begin{align*}
    &\text{Mode= }  \arg \max_r g_R(r\mid \underline X), \\ 
    &\text{Median= } r_m:\int_{0}^{r_m}g_R(r\mid \underline X)\text{d} r = 0.5, \\
    &\text{Mean= } \int_0^{+\infty}r g_R(r\mid \underline X)\text{d} r.
\end{align*}
The mode of a Gamma distribution is null when $\widehat{\theta}$ has a value lower than one. The mode will therefore not be a good indicator of the posterior frailty density because one may observe several zero values across the families. We therefore exclude it as a valid indicator. On the hand, from the formulas of the Gamma distribution, the mean is immediately obtained as \begin{align*}
    \widehat{r}_i = \frac{d_i + \widehat{\theta}}{\widehat{\theta} + \widehat{\Lambda}_0(\underline x_i)}
\end{align*} which, following \cite{balan2019frailtyem}, can be rewritten as \begin{align*}
    \widehat{r}_i = w_i\frac{d_i}{\widehat{\Lambda}_0(\underline x_i)} + (1-w_i),
\end{align*} where $w_i = \dfrac{\widehat{\Lambda}_0(\underline x_i)}{\widehat{\theta} + \widehat{\Lambda}_0(\underline x_i)}$ is the weight of the weighted average between the ratio of the observed events $d_i$ on the sum of the cumulative hazard by families $\widehat{\Lambda}_0(\underline x_i)$ 
% that can be interpreted as the expected number of events 
and one, the prior frailty mean. The median is instead computed numerically quite easily from the estimated family-specific posterior frailty risk distribution.

% One may consider as an alternative, the lower-bound of the highest posterior density \cite{hyndman1996computing}. This quantity, given a desired confidence level $(1-\alpha)\%$, is given by:  \[
% \widehat{r} = \underset{r}{argmin} \left(f(r)\ge f_\alpha\right).
% \] where $f_\alpha$ is the lower-bound of the density function such that the higher posterior density is the interval with area under the curve of (1-$\alpha$). However, this indicator is again not too useful because the lower-bound of each family interval is close to zero. As a consequence, one may not easily distinguish between the low and the high-risk families, which is the ultimate goal. 

Because the posterior mean does not really have a scale of reference, we introduce also another meaningful summary of the posterior frailty risk distribution< i.e. the posterior probability of belonging to the highest-risk group by computing the probability that the family-specific risk value is greater than a fixed value from the prior distribution. By setting a percentile value $\alpha$, the posterior probability of belonging to the highest-risk group is \begin{align*}
    P\left(R \ge F^{-1}_R\left(\alpha; \widehat{\theta}\right)\mid \text{family data}; \widehat{\theta}\right), 
    % = 1 - F_{R_\text{post}}(\widehat{r}\mid \text{family}; \widehat{\theta})
 \end{align*}
where $F^{-1}_R\left(\alpha; \widehat{\theta}\right)$ is the estimated prior quantile associated to a probability equal to $\alpha$ on the left side. In the real case study in Section \ref{sec:1.6} we choose an $\alpha = 0.05$, and we compute the corresponding quantile. Once we have the binary classification of families into two risk groups (low-risk and highest-risk group) based on their probability of belonging the the highest-risk group, we can validate it through the receiver operating characteristic (ROC) curve. We can extract the area under the curve (AUC), the positive predictive value (PPV) and the negative predictive value (NPV) from the ROC curve. These measures can assess the accuracy in binary splitting applied by the models. Notice that this validation procedure can be applied only in the case of simulation studies, since the true family-specific risk needs to be known.

\subsection{Parametric scenario}\label{sec:1.4.2}
On the contrary to the semiparametric scenario, in the parametric scenario we need to specify a distribution for the baseline survival function. The advantage is that the baseline survival function is directly available by estimating its distribution parameters. This allows one to obtain the posterior mean, mode, and median from the parametric distribution of the posterior risk frailty. Also, the extension to predict the posterior risk frailty for a new family in the dataset in straightforward. The family-specific posterior frailty risk distribution given the whole family data is given by \begin{align*}
    &g_R(r\mid \underline x) = \dfrac{f(\underline x\mid r)g_R(r)}{f(\underline x)} = \dfrac{f(x_1\mid r)\dots f(x_{n_i}\mid r)g_R(r)}{\int_{\mathbb{R}^+}f(x_1\mid r)\dots f(x_{n_i}\mid r)g_R(r)\text{d}r} \\
    &=\dfrac{\prod_{j=1}^{n_i}\left(f_r(x_j)^{\delta_j} S_r(x_j)^{1-\delta_j}\right)g_R(r)}{\int_{\mathbb{R}^+}\prod_{j=1}^{n_i}\left(f_r(x_j)^{\delta_j} S_r(x_j)^{1-\delta_j}\right)g_R(r)\text{d}r} = \\
    &=\dfrac{\prod_{j=1}^{n_i}\cancel{\left((1-p)\widetilde f(x_j)\right)^{\delta_j}}\left(p + (1-p)\widetilde S(x_j)\right)^{(\cancel{r-1)\delta_j}+r(1-\cancel{\delta_j})} r^{\sum_{j=1}^{n_i}\delta_j}g_R(r)}{\prod_{j=1}^{n_i}\cancel{\left(\dfrac{(1-p)\widetilde f(x_j)}{p + (1-p)\widetilde S(x_j)}\right)^{\delta_j}}\dfrac{\Gamma(\alpha + \sum_{j=1}^{n_i}\delta_j)}{\Gamma(\alpha)\beta^{\sum_{j=1}^{n_i}\delta_j}}}\left(1 - \dfrac{\text{log}(H(x_j;p,\underline\gamma))}{\beta}\right)^{\alpha + \sum_{j=1}^{n_i}\delta_j} \\
    &=\dfrac{\cancel{\Gamma(\alpha)}\beta^{\sum_{j=1}^{n_i}\delta_j}}{\Gamma(\alpha + \sum_{j=1}^{n_i}\delta_j)}\left(1 - \dfrac{\text{log}\prod_{j=1}^{n_i}(p + (1-p)\widetilde S(x_j))}{\beta}\right)^{\alpha + \sum_{j=1}^{n_i}\delta_j}\cdot \\
    &\cdot\prod_{j=1}^{n_i}\left(p + (1-p)\widetilde S(x_j)\right)^r
    r^{\sum_{j=1}^{n_i}\delta_j}\dfrac{\beta^\alpha}{\cancel{\Gamma(\alpha)}}r^{\alpha-1}\text{e}^{-\beta r} \\
    &=\left(1 - \dfrac{\text{log}\prod_{j=1}^{n_i}(p + (1-p)\widetilde S(x_j))}{\beta}\right)^{\alpha + \sum_{j=1}^{n_i}\delta_j}\prod_{j=1}^{n_i}\left(p + (1-p)\widetilde S(x_j)\right)^r\dfrac{\beta^{\alpha + \sum_{j=1}^{n_i}\delta_j}}{{\Gamma(\alpha + \sum_{j=1}^{n_i}\delta_j)}}r^{\alpha + \sum_{j=1}^{n_i}\delta_j-1}\text{e}^{-\beta r} \\
    &= \left(\dfrac{\beta-\log\prod_{j=1}^{n_i}(p + (1-p)\widetilde S(x_j))}{\cancel{\beta}}\right)^{\alpha + \sum_{j=1}^{n_i}\delta_j}\dfrac{\cancel{\beta^{\alpha + \sum_{j=1}^{n_i}\delta_j}}}{{\Gamma(\alpha + \sum_{j=1}^{n_i}\delta_j)}}r^{\alpha + \sum_{j=1}^{n_i}\delta_j-1}\text{e}^{-(\beta -\sum_{j=1}^{n_i}\log\left(p + (1-p)\widetilde S(x_j)\right))r} \\
    &\text{which is distributed as a Gamma}\left(shape = \alpha + \sum_{j=1}^{n_i}\delta_j, rate = \beta -\sum_{j=1}^{n_i}\log\left(p + (1-p)\widetilde S(x_j)\right)\right).
\end{align*} 

The density function, in the univariate case, reduces immediately to \begin{align*}
    &g_R(r\mid \underline x)=\left(1 - \dfrac{\text{log}(p + (1-p)\widetilde S(x))}{\beta}\right)^{\alpha + \delta}\left(p + (1-p)\widetilde S(x)\right)^r\dfrac{\beta^{\alpha + \delta}}{{\Gamma(\alpha + \delta)}}r^{\alpha + \delta-1}\text{e}^{-\beta r},
\end{align*} 
which is distributed as a Gamma$\left(shape = \alpha + \delta, rate =\beta - \log(p + (1-p)\widetilde S(x))\right)$.

The posterior mean and mode frailty are extracted from the formula associated to the Gamma distribution as \begin{align*}
    \text{mean} = \dfrac{\alpha + \sum_{j=1}^{n_i}\delta_j}{\beta - \sum_{j=1}^{n_i}\log\left(p + (1-p)\widetilde S(x_j)\right)}, \qquad
    \text{mode} = \dfrac{\alpha + \sum_{j=1}^{n_i}\delta_j - 1}{\beta - \sum_{j=1}^{n_i}\log\left(p + (1-p)\widetilde S(x_j)\right)}.
\end{align*} 

The univariate case is directly obtained by setting family size $n_i=1$. Note that when the shape parameter of the distribution is lower than one, the mode is not a reliable summary of the posterior frailty risk. Moreover, the posterior median of a Gamma distributed frailty risk requires numerical methods to be computed as there is not a formula in closed form. For these reasons the posterior mean appears to be the most reliable and easily to compute.  

To validate that the mean is the best performing as posterior risk, the mean, median, and mode are compared in terms of various indices. When the analysis in on simulated data, we can compare the true frailty risk from the data generating step $r_i$ to the predicted frailty risk $\widehat r_i$, for $n$ families. For example, we compute the estimated mean squared prediction error (MSPE), whose equation is given by \begin{align*}
    MSPE = \dfrac{1}{n}\sum_{i=1}^n(\widehat r_i - r_i)^2.
\end{align*}
Also, from the linear regression of the true frailty risk $r_i$ on the predicted frailty risk $\widehat r_i$, we report the estimated coefficient of determination $R^2$ as a measure of how much the predicted risk can explain the variability of the true risk. The coefficient of the predicted frailty risk, the linear correlation coefficient of Pearson and the rank correlation coefficient are also extracted and reported. 

Once we prove that the mean is the best performing posterior risk summary, also the models under analysis need to be validated in terms of prediction accuracy, depending on whether the analysis is on simulated studies or on the real case.

When the analysis is on simulated studies, we compute the AUC for the binary splitting and the Harrell's concordance index. We choose this index because it can be computed also in the real case application, as it is a measure of model performance for unknown outcomes \cite{rahman2017review, harrell1982evaluating, scheike2015estimating, tokatli2011developing}, and it makes the comparison between simulation study and real case much easier. This index estimates the probability that, among a randomly selected pair of subjects with different right-censored survival times and predicted risk values, the one with a lower risk score will outlive the other \cite{harrell1982evaluating}. This define this pair as ``concordant''. Notice that this is similar to what is done in the modified version of the Wilcoxon test on survival data with right censoring.

The estimate relies on the ratio of concordant pairs to all pairs, which includes discordant and tied pairs. The formula is given by \cite{schmid2016use}
\[
C = \dfrac{\sum_{i\ne j}\mathbb{I}(\widehat{r}_i < \widehat{r}_j)\mathbb{I}(x_i > x_j)\delta_j}{\sum_{i\ne j}\mathbb{I}(x_i > x_j)\delta_j},
\] where $\widehat{r}_i$ and $\widehat{r}_j$ denote the predicted risk for two subjects $i$ and $j$ who are not members of the same family. It is of interest to notice that from theoretical result the Harrell's concordance index does not depend on the baseline survival function, but only on a random variable $Y^*\sim\text{Beta}(\theta + 1,\theta)$, such that \[
C(\theta) = S_{Y^*}\left( \frac{1}{2} \right).
\]
An extended version of this result can be found in Appendix \ref{appendix:1.c}.

\section{Simulation study}\label{sec5}
We present results from the comparative analysis on the six models we developed or included from literature. In Table \ref{tab:models} we present the six models highlighting their commonalities and differences.

\begin{center}
\begin{table*}[!h]%
\begin{tabular*}{\textwidth}{@{\extracolsep\fill}lllll@{}}
    \toprule
    Name of the model &Likelihood &Frailty risk &FH &Survival  \\ \midrule
    Multivariate frailty Cure-Rate &multivariate &\cmark &\xmark &cure-rate \\
    Univariate frailty Cure-Rate &univariate &\cmark &\xmark &cure-rate \\ 
    Univariate $FH$ Cure-Rate &univariate &\xmark &constant &cure-rate \\
    Multivariate frailty Cox &multivariate &\cmark &\xmark &traditional \\
    Univariate $FH$ Cox &univariate &\xmark &constant &traditional \\
    Univariate $FH(t)$ Cox &univariate &\xmark &time-varying &traditional
    \\ \bottomrule
    \end{tabular*}
    \caption{Commonalities and differences among the six models compared.}
    \label{tab:models}
\end{table*}
\end{center}

Notice that the Cure-Rate models we develop, i.e. the Multivariate frailty Cure-Rate, the Univariate frailty Cure-Rate, and the Univariate $FH$ Cure-Rate model are parametric and thus in the analysis we explore several baseline survival distributions to be involved in the parametric models, including Weibull, Gamma, Lognormal, three-parameter Gamma, also known as the Pearson type III distribution (see, e.g., \cite{bobee1991risk}), and three-parameter Lognormal. These two last distributions are included to assess whether an increased flexibility in the distribution could benefit both inference precision and prediction accuracy. 

The section is organized starting with the data generating process, above all for the cure-rate structure, and then progressing to the fitting of semiparametric models and then parametric models. We need to make a distinction between them because only in the parametric scenario we analyse different time-to-event distributions. All over the section we compare the model goodness of fit, and their predictive accuracy through the AUC, PPV, NPV and Harrell's Concordance index.

\subsection{Data generating process from the Lehmann Cure-Rate model}

A critical aspect to consider when generating data from a cure-rate survival model \cite{mota2022new} is that one needs to employ distinct methods for generating survival times for cases and cured observations. 

Recall that the model takes the Lehmann form
\[ S(t; r) = \left[ S_0(t) \right]^r {\rm,\  with\ } S_0(t)=\left[ p_0+(1-p_0) \widetilde{S}(t) \right], \ i.e.\ \alpha(r)=r \ {\rm for}\  r >0. \]

To generate survival times from this model (with a set of parameter values with varying lenght), one may proceed as follows:
\begin{enumerate}
\item generate a family-specific (or subject-specific if the model is univariate) frailty risk term $r$ from the Gamma$(shape = \theta, rate = \theta)$ distribution;
\item generate a subject-specific value $u$ from the Unif$(0,1)$ distribution;
\item {\sf if} $u \leq p_0^r$
\begin{itemize}
\item[] {\sf then} set $t = + \infty$ (a cured observation);
\item[] {\sf else} solve $u = \left[ p_0 + (1-p_0) \widetilde{S}(t)  \right]^r$, which yields immediately
\[ t = \widetilde{S}^{-1} \left( \frac{u^{1/r} - p_0}{1-p_0} \right)=\widetilde{F}^{-1} \left( \frac{1-u^{1/r}}{1-p_0} \right), \]
 with $\widetilde{F}^{-1}( \cdot)$ the quantile function of the time-to-event distribution. Since we explore several distributions, we can easily find the two-parameter distributions in the basic \texttt{R} software, while the package \texttt{FAdist}, still from the \texttt{R} software, contains functions for the cumulative distribution function (CDF), density function, quantile function, and random sample function from the three-parameter Gamma and three-parameter Lognormal. 
\end{itemize}
\end{enumerate}
Independent (typically, administrative) right censoring of all observations can be applied to obtain the observed data. Censoring is the minimum between the date of death and the end year of the follow-up, set at 2020. For building the follow-up we need to fix a starting point for all the individuals, which is date of birth. We start from generating the mother date of birth from a Uniform between 1905 and 1945. The daughter dates of birth are generated summing up to their mother birth date a value from a Uniform between 25 and 35, which coincides to the age the mother gave birth. If one takes into account the fact that family members from different generations have different birth dates (hence in calendar time), follow-up is clearly longer for members born earlier. Date of death is generated similarly keeping a time coherence between mothers and daughters. 

In the multivariate scenario, the data generating follows easily from the conditional independence assumption within each family, assuming for all family members the same frailty risk $r$. Also, in this scenario is crucial to random generate the size of families included in the sample. To this end, we randomly generate family size as one plus the minimum of a fixed number ($n_F-1$) and a value generated from a Poisson random variable with parameter $\lambda_F$. To give an insight, one can see $n_F$ as the maximum family size all over the sample, and $\lambda_F$ the average number of first-degree female relatives a subject can have. 

The parameters involved in the models are set to $p = 0.85$, $shape = 8$, $scale = 6$, $threshold = 15$, $\theta = 0.2$ where, recall, $p$ is the cured fraction, $shape$, $scale$, $threshold$ are the three parameters of the baseline survival function on cases. The threshold parameter is only used for a three-parameter distribution. And, $\theta$ is the frailty risk parameter. Also, notice that when we employ the Lognormal distribution the parameters are called differently $\mu = 6$ and $\sigma^2 = 8$, replacing respectively $shape$ and $scale$.

On the other hand, if a traditional survival function is involved, the data generation would typically be simpler. For example, if $S_r(t)=[S_0(t)]^r$ with $S_0(t)$ the traditional survival function of the Weibull($k,\lambda$) random variable, it is immediate to check that $T\mid R = r\sim$Weibull$\left( k, \lambda\sqrt[k]{r}\right)$. Usually, a reparametrization dependent on the frailty risk is sufficient to obtain the updated distribution. 

\subsection{Semiparametric scenario}
Among the six models we compare, three of them are semiparametric: the Multivariate frailty Cox models, the Univariate $FH$ Cox model and the Univariate $FH(t)$ Cox model. 

In Table \ref{tab:1.1} we compare the models in terms of recovering the true parameter values set in the data generating process. We generate a sample of 100,000 families, with family size varying between 1 and $n_F=5$, with a mean of $\lambda_F = 0.8$ relatives. Our experience suggests that a sample size of 100,000 families is sufficient to ensure accurate results with reasonable computational speed. The simulation is repeated for 100 times. 

Parameter estimation is performed through an EM algorithm, that can be carried out using functions from the \texttt{R} package \texttt{frailtyEM} \cite{balan2019frailtyem, balan2020tutorial}. However, the functions of the \texttt{frailtyEM} package are time and memory-consuming for large datasets. In contrast to the approach commonly found in the literature, we speed up this process by utilizing the \texttt{cox} function in the \texttt{survival} package. This allows us to efficiently obtain all the necessary quantities within a reasonable time and without excessive memory usage. 
\begin{center}
\begin{table*}[!h]%
\begin{tabular*}{\textwidth}{@{\extracolsep\fill}lllll@{}}
\toprule
    &$\theta$ &mean($\widehat{\theta}$) (se) &mean($\widehat{\beta}_{FH}$) (se) &mean(C) (se) \\ \cmidrule{2-5} 
    Multivariate frailty Cox &0.2 &0.199 (0.004) &- &0.966 (<0.001) \\
    Univariate $FH$ Cox &&- &2.835 (0.081) &0.542 (0.002) \\ 
    Univariate $FH(t)$ Cox &&- &5.846 (0.166) &0.556 (0.002) \\ \midrule
    Multivariate frailty Cox &0.5 &0.499 (0.012) &- &0.949 (0.001) \\ 
    Univariate $FH$ Cox &&- &1.455 (0.040) &0.514 (0.001) \\
    Univariate $FH(t)$ Cox &&- &2.956 (0.083) &0.533 (0.001)  \\ \midrule
    Multivariate frailty Cox &0.8 &0.802 (0.029) &- &0.943 (0.001) \\ 
    Univariate $FH$ Cox &&- &1.102 (0.033) &0.503 (0.001) \\
    Univariate $FH(t)$ Cox &&- &2.227 (0.066) &0.523 (0.001) \\ \bottomrule
    \end{tabular*}
    \caption{Mean and standard error of the estimated parameters and Harrell's Concordance index (C) in the last column. The first column $\theta$ stands for the true value set at the data generating step and varying among (0.2, 0.5, 0.8).}
    \label{tab:1.1}
\end{table*}
\end{center}
% codice simulation-cox-contfrailty-multiFhCostantAndTimevar-MVV.R LOCAL COMPUTER

Results from Table \ref{tab:1.1} show that the Multivariate frailty Cox model can accurately estimate the value of $\theta$ from the given data, while the Univariate models only estimate the parameter $\beta_F$ and nothing can be said about parameter recovery. Even though the parameter $\beta_F$ has little meaning because is not part of the data generating process, it is interesting to notice is always estimated positive. This means that the family history is a valid and positive indicator for breast cancer development.

Also, in Table \ref{tab:1.1}, the Harrell's Concordance index is reported. Notice that the value of the Harrell's Concordance index depends on the discrete nature of the predicted risk: for example, if a model predicts only two levels of risk, then the numerator of the index will tend to be small, as all tied pairs $(i,j)$ for which $\widehat r_i = \widehat r_j$ are excluded. This may happen with the univariate models which include $FH$ and $FH(t)$. Results show that the Multivariate frailty Cox model is, not surprisingly, the best among the models, as it is used to generate the data. Given its benchmark Concordance index value of about 0.95, the Univariate models have a poor Concordance performance around only the 50\% of pairs $(i,j)$ to be concordant in terms of time-to-event and risk values. In Table \ref{tab:1.1} the best result by true value of $\theta$ is in bold, while the absolute best among all the scenario is highlighted). 

% \begin{table}[ht] 
%     \centering
%     \begin{tabular}{llll} 
%     \hline 
%     \multicolumn{4}{c}{Multivariate model} \\ 
%     Distribution & $\widehat{\theta}$ (sd) &$\widehat{\beta}_{FH}$ (sd) &Concordance (sd) \\ \hline
%     Weibull &0.2 (0.001) &- &0.9657 (0.0001) \\ 
%     Gamma &0.1997 (0.002) &- &0.9734 ($1\cdot 10^{-4}$) \\
%     LogNormal &0.2003 (2.6e-03) &- &\textbf{0.9844} (5.9$\cdot 10^{-5}$) \\ 
%     Gamma 3-pars &$0.2004 (1.6\cdot 10^{-3})$ &- &0.9775 ($1\cdot 10^{-4}$) \\
%     LogNormal 3-pars &0.204 (0.002) &- &0.9808 ($1\cdot 10^{-4}$) \\ \hline
%     \multicolumn{4}{c}{Univariate model} \\ \hline
%     Weibull &4645389 (5641495) &- &- \\ 
%     Gamma &19162269 (7440993) &- &- \\
%     LogNormal &23296416 (10564719) &- &- \\ 
%     Gamma 3-pars &28463849 (6068295)  &- &- \\
%     LogNormal 3-pars &21169218 (10640070) &- &- \\ \hline
%     \multicolumn{4}{c}{Observed family history model} \\ \hline
%     Weibull &- &3.8481 (0.0472) &0.5849 (0.0012) \\ 
%     Gamma &- &2.8470 (0.0249) &0.5497 (0.0006) \\
%     LogNormal &- &2.4601 (0.0565) &0.5283 (0.0011) \\ 
%     Gamma 3-pars &- &2.4999 (0.0295) &0.5381 ($5\cdot 10^{-4}$) \\
%     LogNormal 3-pars &- &2.7289 (0.0322) &0.5386 (0.0006) \\ \hline
%     \end{tabular}
%     \caption{Estimated parameters and concordance index from the analysed models.}
%     \label{tab:1}
% \end{table}

Our focus now shifts towards the core of the study, which is frailty prediction. In this regard, the algorithm developed in Section \ref{sec4} is employed to estimate the risk of developing breast cancer with the current dataset and to predict the risk for a new subject. The mean risk and median risk are predicted for each family. In terms of the Multivariate frailty Cox model, these two summary measures are compared in terms of MSPE, $R^2$ from the linear regression of the true risk on the predicted risk, the coefficient $\widehat{\beta}$ of the predicted risk, and both the Pearson's and the rank correlation coefficient. Clearly, this is not possible for the Univariate models since they compare the estimated family history to the true risk. The comparison is on a sample of 100,000 families and varying family size among $(n_{F}, \lambda_{F}) = ((2,0.8), (5,0.8), (10,5), (20,10))^T$, since differences emerge when family size increases. Average and standard error of the estimated measures are reported. Results in Table \ref{tab:1.2} show that higher is the average family size then higher are the performances of both mean and median risk, meaning that the family size affects in a positive way the risk prediction accuracy, as expected. 

Results show that the median performs poorly as a risk predictor, especially when family size is small. However notice that, as the family size decreases, the mean is known to exhibit a shrinkage problem due to the low number of events occurring within families. Figure \ref{fig:scatterquantile} illustrates this phenomenon, in which the predicted mean risk do not reach zero when the true risk is close to zero, and all values are shrunken towards the true (prior) mean of one. Notice that the second plot of Figure \ref{fig:scatterquantile}, resulting from the maximum family size equal to twenty, has a lower shrinkage compared to the first plot which has a maximum family size equal to five. This well-known phenomenon is due to the fact that a large family size is required to increase the sum of cumulative hazards on observed time and allow the mean to shrink towards zero \cite{balan2019frailtyem, balan2020tutorial}. One should keep in mind this limitation of the posterior frailty risk mean when running risk prediction. For large dataset, the shrinkage effect becomes smaller and smaller towards zero and thus we do not encounter this problem on the real case data because of its high dimension.

\begin{center}
\begin{table*}[!h]%
\begin{tabular*}{\textwidth}{@{\extracolsep\fill}lllll@{}}
\toprule
    &\multicolumn{4}{c}{($n_F, \lambda_F$)} \\\cmidrule{2-5} 
    % &(1, 0.8) 
    &(2, 0.8) &(5, 0.8) &(10, 5) &(20, 10) \\ \midrule % &(50, 10)
    Multivariate \\ 
    MSPE$_\text{mean}$ &3.78(<0.01) &3.68(<0.01) &\hl{\textbf{2.88(<0.01)}} &3.44(<0.01) \\ % &3.51(<0.01) \\
    MSPE$_\text{median}$ &5.77(<0.01) &5.85(<0.01) &4.88(<0.01) &\hl{\textbf{4.00(<0.01)}} \\ % &\textbf{3.92(<0.01)} \\ 
    $R^2_\text{mean}$ &0.23(<0.01) &0.25(<0.01) &0.51(<0.01) &\hl{\textbf{0.65(<0.01)}} \\ %&\textbf{0.65(<0.01)} \\
    $R^2_\text{median}$ &0.16(<0.01) &0.16(<0.01) &0.25(<0.01) &\hl{\textbf{0.29(<0.01)}} \\ %&\textbf{0.29(<0.01)} \\
    $\widehat{\beta}_{\text{mean}}$ &\hl{\textbf{0.94(<0.01)}} &0.91(<0.01) &0.72(<0.01) &0.61(<0.01) \\ % &0.60(<0.01) \\
    $\widehat{\beta}_{\text{median}}$ &11.81(0.01) &9.03(0.01) &2.99(<0.01) &\hl{\textbf{1.45(<0.01)}} \\ % &\textbf{1.44(<0.01)} \\ 
    $\rho_\text{mean}$ &0.49(<0.01) &0.54(<0.01) &0.74(<0.01) &\hl{\textbf{0.81(<0.01)}} \\ % &\textbf{0.82(<0.01)} \\
    $\rho_\text{median}$ &0.42(<0.01) &0.42(<0.01) &0.52(<0.01) &\hl{\textbf{0.55(<0.01)}} \\ % &\textbf{0.55(<0.01)}  \\
    Rank $\rho_\text{mean}$ &0.24(<0.01) &0.28(<0.01) &0.48(<0.01) &\hl{\textbf{0.58(<0.01)}} \\ % &\textbf{0.58(<0.01)} \\ 
    Rank $\rho_\text{median}$ &0.11(<0.01) &0.15(<0.01) &0.36(<0.01) &\hl{\textbf{0.49(<0.01)}} \\ % &0.49(<0.01) \\ 
    \midrule
    Univariate FH \\
    MSPE &5.87(<0.01) &5.97(<0.01) &5.47(<0.01) &\textbf{5.36(<0.01)} \\ % &5.27(<0.01)  \\
    $R^2$ &0.03(<0.01) &0.03(<0.01) &0.13(<0.01) &\textbf{0.16(<0.01)} \\ % &0.16(<0.01) \\
    $\widehat{\beta}_{\text{FH}}$ &3.48(<0.01) &3.29(<0.01) &2.99(<0.01) &\textbf{2.71(<0.01)} \\ % &2.71(<0.01) \\ 
    $\rho$ &0.17(<0.01) &0.19(<0.01) &0.36(<0.01) &\textbf{0.39(<0.01)} \\ % &0.39(<0.01)  \\ 
    Rank $\rho$ &0.14(<0.01) &0.16(<0.01) &0.35(<0.01) &\textbf{0.43(<0.01)} \\ % &0.43(<0.01) \\ 
    \midrule 
    Univariate FH(t) \\ 
    MSPE &2.32(0.0067) &2.3(0.0065) &\textbf{2.08(0.006)} &2.22(0.0064) \\ 
    $R^2$ &0.03(0.0004) &0.04(0.0003) &\textbf{0.05(0.0002)} &0.04(0.0001) \\ 
    $\widehat\beta_F$ &\textbf{0.91(0.0062)} &0.87(0.0049) &0.62(0.0017) &0.47(0.001) \\ 
    $\rho$ &0.18(0.0011) &0.19(0.0009) &\textbf{0.23(0.0004)} &0.19(0.0004) \\ 
    Rank $\rho$ &0.18(0.0008) &0.2(0.0006) &\textbf{0.28(0.0004)} &0.25(0.0004) \\ 
  \bottomrule
    \end{tabular*}
    \caption{Prediction accuracy measures for family size defined by the two parameters ($n_F, \lambda_F$) among the values (2, 0.8), (5, 0.8), (10, 5), (20, 10); $n_F$ indicates the largest number of family members, and $\lambda_F$ represents the parameter of the Poisson distribution used to generate the additional number of family members apart the main subject.}
    \label{tab:1.2}
\end{table*}
\end{center}
% Code from postPrediction3.R on SWEDISH COMPUTER. 
\newpage
\begin{figure}
    \centering
    \includegraphics[width = \linewidth]{plots/Rplot43.pdf}
    \includegraphics[width = \linewidth]{plots/Rplot47.pdf}
    \caption{True frailty risk versus predicted frailty risk mean with the bisector of the third quadrant (in red), the smoothed regression line (in green with confidence interval) and smoothed quantile regression line on the first, second and third quantile (in orange) for a smaller and higher family size, respectively starting from the top, equal to a maximum of 5 and 20.}
    \label{fig:scatterquantile}
\end{figure}
% Code from postPredBackup3.R in SWEDISH COMPUTER varying the sample size (in terms of families) and the internal family size parameters.

Also, another limitation of the posterior frailty risk mean is the variability in the prediction. The mean squared prediction error (MSPE) is quite high to provide an accurate estimate of the risk of breast cancer occurrence in a family. We explore the values of the root MSPE conditional on the value of the true frailty risk in Figure \ref{fig:cspe}. The MSPE increases as the true frailty risk increases too, regardless of the family size. We observe a possible flattening of the curve with a larger family size in the second plot of Figure \ref{fig:cspe}, but the prediction error remained high. This could lead to accurately identify the lower-risk families but not the higher-risk families. This means that we may encounter a large number of false negative cases. 

\begin{figure}
    \centering
    \includegraphics[width = \linewidth]{plots/Rplot44.pdf}
    \includegraphics[width = \linewidth]{plots/Rplot48.pdf}
    \caption{True frailty risk versus the conditional MSPE in squared root, for a lower and a higher family size, respectively starting from the top, equal to a maximum of 5 and 20.}
    \label{fig:cspe}
\end{figure}
% Code from postPredBackup3.R

Given the structural limitations of the posterior mean frailty that we report so far, we suggest an alternative approach. Specifically, we compute the family-specific posterior probability of belonging to the group of the highest-risk families as anticipated in Section \ref{sec4}. This is achieved by calculating the probability of the posterior frailty (mean, median, mode) exceeding the $(1-\alpha)$ percentile (say 95\% below) of the frailty population, or say prior, Gamma distribution. A plot of the the true prior versus the estimated posterior probability of belonging to the high-risk families is reported in Figure \ref{fig:1.3}, assessing that the relation between the true and the estimated probability of belonging to the highest-risk family is coherent, i.e. the posterior probability increases when the prior probability increases.

Following some calculation, we can use a classifier to identify the membership in the extreme cluster of families with the same threshold, where the indicator is equal to one whenever the family-specific probability is at least equal to 15\%, that is the $95$th quantile in the posterior distribution of the population probability. Also, notice from Figure \ref{fig:1.3} that we can distinguish two clouds of points, where the highest recall the highest-risk families. 

Therefore, after all the analysis run, we adopt the mean as the best posterior frailty risk summary and thus we perform final computations only with the Multivariate frailty Cox model.

The accuracy of the binary classification can be analyzed using the classical area under the ROC curve (AUC), the positive predictive value (PPV), and the negative predictive value (NPV), which are all measures associated with the binary diagnostic testing. The AUC, PPV, and NPV are presented in Table \ref{tab:1.3} and indicate that all measures improve as the family size increases, as expected. This can be appreciated also from the ROC curve in Figure \ref{fig:1.4}, where the case with the higher family size with a maximum of twenty correctly classifies the 97\% of families in the right group, the 15\% above the case with maximum five family members. However, all the results are promising as shown in Table \ref{tab:1.3}, since the AUC is in the range 0.7-0.85: this indicates that around the 70\%-85\% of families are correctly classified in the right category low-risk or high-risk also with small family sizes. Now, the negative predictive value (NPV) is always significantly higher than the positive predictive value, meaning that to the higher value of the AUC the NPV contributes more. 

Prediction accuracy is determined through an in-sample validation, without distinguishing between a training and a test set. 

Figure \ref{fig:1.3} displays a scatter-plot that depicts the relationship between the true frailty percentiles into the population (prior) distribution and the posterior probability of belonging to the highest-risk families, with the threshold set at 95$\%$. The threshold for splitting families into two groups can be easily changed and run again the split, but we chose 95$\%$ because we wanted to identify the highest 5$\%$ risk group. From this plot we can appreciate how increasing the family size in the sample the highest-risk families can be more identified by noting a more distinguished point cloud, and a more inclined smoothed line indicating the tendency of the entire cloud. It is interesting to notice that we can identify two risk groups. 
\begin{figure}[ht]
    \centering
    \includegraphics[width=\linewidth]{PhD_thesis_template/plots/Rplot104.pdf}
    \includegraphics[width=\linewidth]{PhD_thesis_template/plots/Rplot103.pdf}
    \caption{True frailty percentiles into the population (prior) distribution and the posterior probability of belonging to the highest-risk families, with a 95$\%$ threshold.}
    \label{fig:1.3}
\end{figure}
\newpage
\begin{center}
\begin{table*}[!h]%
\begin{tabular*}{\textwidth}{@{\extracolsep\fill}lllll@{}}
\toprule
    &\multicolumn{4}{c}{($n_F, \lambda_F$)} \\ \cmidrule{2-5}
    &(2, 0.8) &(5, 0.8) &(10, 5) &(20, 10) \\ \midrule % &(50, 10) \\ 
    Multivariate \\
    AUC &0.80 (0.0001) &0.83 (0.0001) &0.94 ($<$0.0001) &\hl{\textbf{0.97 ($<$0.001)}} \\ % &0.97($<$0.001) \\ 
    PPV &0.39 (0.0002) &0.41 (0.0002) &0.56 (0.0002) &\hl{\textbf{0.63 ($<$0.001)}} \\ %&0.63($<$0.001)  \\
    NPV &0.97 ($<$0.0001) &0.97 ($<$0.0001) &\textbf{0.98 ($<$0.0001)} &\textbf{0.98 ($<$0.001)} \\ % &0.98($<$0.001) \\ 
    Concordance \\
    \midrule
    Univariate $FH$ \\ 
    AUC  &0.53 (0.0001) &0.54 (0.0001) &0.66 (0.0001) &\textbf{0.70 ($<$0.001)} \\ % &0.70($<$0.001) \\ 
    PPV &\textbf{0.32 (0.0004)} &0.29 (0.0003) &0.25 (0.0001) &0.21 ($<$0.001) \\ % &0.21($<$0.001) \\
    NPV &\hl{\textbf{0.99 ($<$0.0001)}} &\hl{\textbf{0.99 ($<$0.0001)}} &0.94 ($<$0.0001) &0.90 ($<$0.001) \\ % &0.90($<$0.001) \\ 
    Concordance \\ 
    \midrule
    Univariate $FH(t)$ \\ 
    AUC &0.55 (0.0004) &0.56 (0.0004) &0.59 (0.0003) &0.57 (0.0002) \\ 
    PPV &0.27 (0.0013) &0.24 (0.0012) &0.16 (<0.0001) &0.14 (0.0001) \\ 
    NPV &0.91 (0.0004) &0.88 (0.0005) &0.52 (0.0006) &0.36 (0.0005) \\ 
    Concordance \\ 
    \bottomrule
    % \multicolumn{6}{l}{1000000 families} \\ \hline
    % AUC &0.7027 &0.7112 &0.7133 &\textbf{0.8257} &0.8233  \\
    % Positive Predictive Value &0.348 &0.3485 &0.3513 &\textbf{0.6126} &0.6120 \\
    % Negative Predictive Value &0.9704 &0.9713 &0.9717 &\textbf{0.9776} &\textbf{0.9776} \\ \hline
    \end{tabular*}
    \caption{AUC, PPV and NPV corresponding to the ROC curve on the binary frailty risk with threshold (1 - $\alpha) = 0.95$. Family size is varying with the values ($n_f, \lambda_F$), on 100 simulations on 100,000 families.}
    \label{tab:1.3}
\end{table*}
\end{center}
% Complete code from postPredBackup3.R or
% AUC for (5,0.8) - but anyway every change of parameters can be done - is in the code \texttt{simulation-cox-contfrailty-multFhConstAndTimevar-postprediction-MVV}. 
In conclusion, the Multivariate Shared Frailty Cox model appears to be an effective approach to the study of family history effects and risk prediction. Once the posterior frailty distribution is obtained for each family, we recommend to report the posterior frailty mean as well as the posterior probability and binary indicator of belonging to the highest-risk families based on a threshold from the population (prior) distribution (for example the top 5\% as above).

It should be noted that in the case of the Swedish multi-generational breast cancer dataset, or in general the available data, the binary indicator of belonging to the highest-risk families is not defined only in the case of a population threshold. Indeed, one may also classify a family in relative terms of the highest-risk group of families by a comparison to the empirical distribution of the family-specific posterior probabilities of belonging to that top risks group, across all families in the dataset.
\begin{figure}
% \begin{minipage}{0.5\linewidth}
    \centering
    % \includegraphics[width=\linewidth]{plots/Rplot77.pdf}
    \includegraphics[width=\linewidth]{plots/Rplot75.pdf}
    % \includegraphics[width=0.8\linewidth]{plots/Rplot73.pdf}
    \includegraphics[width=\linewidth]{plots/Rplot71.pdf}
    % \includegraphics[width=\linewidth]{plots/Rplot69.pdf}
    \caption{ROC curve of the predicted indicator of belonging to the highest-risk families versus the true membership to the group of the highest-risk families (top 5\%). Family size is varying with smaller and higher values, with the maximum members per family up to 5 or 20, respectively.}
    \label{fig:1.4}
% \end{minipage}%
% \begin{minipage}{0.5\linewidth}
%     \centering
%     \includegraphics[width=0.8\linewidth]{plots/Rplot86.pdf}
%     \includegraphics[width=0.8\linewidth]{plots/Rplot91.pdf}
%     \includegraphics[width=0.8\linewidth]{}
%     \includegraphics[width=0.8\linewidth]{}
%     \includegraphics[width=0.8\linewidth]{}
%     \caption{ROC curve on one million families of the predicted indicator of belonging to the highest-risk families versus the true membership to the highest-risk families. Family size is varying among the couples of maximum and expected value of the minimum: (2, 0.8), (5, 0.8), (10, 5), (20, 10), (50, 10).}
%     \label{fig:smoothregline}
% \end{minipage}
\end{figure}
% Code from postPredBackup3.R  
% \begin{figure}
% % \begin{minipage}{0.5\linewidth}
%     \centering
%     \includegraphics[width=\linewidth]{plots/Rplot78.pdf}
%     % \includegraphics[width=0.8\linewidth]{plots/Rplot76.pdf}
%     % \includegraphics[width=0.8\linewidth]{plots/Rplot74.pdf}
%     % \includegraphics[width=0.8\linewidth]{plots/Rplot72.pdf}
%     \includegraphics[width=\linewidth]{plots/Rplot70.pdf}
%     \caption{Scatter-plot on 10000 families of the true frailty percentile into the population distribution versus the posterior probability of belonging to the highest-risk families (top 5\%). Family size is varying with the values $(n_f,\lambda_F)$: (2, 0.8), (5, 0.8), (10, 5), (20, 10), (50, 10).}
%     \label{fig:smoothregline}
% % \end{minipage}%
% % \begin{minipage}{0.5\linewidth}
% %     \centering
% %     \includegraphics[width=0.8\linewidth]{plots/Rplot89.jpg}
% %     \includegraphics[width=0.8\linewidth]{plots/Rplot92.jpg}
% %     \includegraphics[width=0.8\linewidth]{}
% %     \includegraphics[width=0.8\linewidth]{}
% %     \includegraphics[width=0.8\linewidth]{}
% %     \caption{Scatter-plot on one million families of the true frailty percentile into the population distribution versus the posterior probability of belonging to the highest-risk families, fixed the percentile at $95\%$. Family size is varying among the couples of maximum and expected value of the minimum: (2, 0.8), (5, 0.8), (10, 5), (20, 10), (50, 10).}
% %     \label{fig:smoothregline}
% % \end{minipage}
% \end{figure}
% probably the code is in postPredBackup3.R SWEDISH COMPUTER but I am not that sure about this. 
\clearpage

\subsection{Parametric models}
Now we explore the Multivariate Shared Frailty parametric model, the Univariate Frailty parametric, and the Univariate model with the constant family history covariate in terms of parameter recovery and accuracy in risk prediction. As we can expect, the Univariate model with the family history covariate can not recover well the parameters because the family history coefficient is not part of the data generating process. Let us start with the analysis. 

Parameter estimation in the parametric case allows for a more detailed description compared to the case of semiparametric estimation, due to the inclusion of a parametric baseline survival function and of the fraction of ``non-cases'' assumed by the cure-rate structure incorporated in the models. We conduct this simulation study on $n=100,000$ families for 100 iterations to obtain average point estimates and their standard errors across repetitions. As above, to generate the family size we use the parameter to control the maximum of the family size $n_F$ and the frailty variance parameter of the Poisson distribution involved $\lambda_F$. For the parameter recovery we set the parameter at $(n_F, \lambda_F) = (5,0.8)$, and the other parameter values are fixed with values reported in the caption of the tables themselves, according to the baseline survival distributions. Indeed, we consider five different baseline survival functions, namely Weibull, Gamma, Lognormal, three-parameters Gamma, and three-parameters Lognormal. In Tables \ref{tab:1.4}, \ref{tab:1.5}, \ref{tab:1.6}, \ref{tab:1.7}, \ref{tab:1.8} we report the true value of the frailty parameters, varying among $(0.2,0.5,0.8)$ in the first column, followed by the baseline survival function parameters recovery, and the frailty parameter recovery in the last column. In bold there are the cases of a good recovery of the true parameter value. 
\begin{center}
\begin{table*}[!h]%
\begin{tabular*}{\textwidth}{@{\extracolsep\fill}llllll@{}}
\toprule
    &$\theta$ &$\widehat p_0$ &$\widehat shape_0$ &$\widehat scale_0$ &$\widehat\theta$ \\ \midrule
    % &0.85 &8 &6 &\textbf{0.2} \\
    Multivariate frailty Cure-Rate \\ 
    Mean &0.2 &0.8500 &7.9974 &5.9997 &\textbf{0.2004} \\
    Se &&0.0016 &0.0498 &0.0063 &0.0043 \\ 
    Mean &0.5 &0.8499 &8.0048 &6.0001 &\textbf{0.5005} \\
    Se &&0.0012 &0.0371 &0.0052 &0.0128 \\
    Mean &0.8 &0.8501 &8.0006 &6.0002 &\textbf{0.8053} \\
    Se &&0.0011 &0.0435 &0.0062 &0.0286 \\ \midrule
    Univariate frailty Cure-Rate \\
    % &0.85 &8 &6 &\textbf{0.2} \\
    Mean &0.2 &0.7914 &8.1218 &6.3699 &0.1109 \\
    Se &&0.0008 &0.0366 &0.0374 &0.0007 \\ 
    Mean &0.5 &0.8138 &8.5950 &6.0571 &0.3868 \\
    Se &&0.0011 &0.0241 &0.0030 &0.0170 \\ 
    Mean &0.8 &0.8145 &8.3576 &6.0688 &0.6187 \\
    Se &&0.0012 &0.0247 &0.0035 &0.0568 \\ \midrule
    &&&&&$\widehat\beta_F$ \\ \midrule
    Univariate FH Cure-Rate \\ 
    Mean &0.2 &0.8961 &5.0588 &6.1471 &2.8782 \\
    Se &&0.0014 &0.0653 &0.0837 &0.0587 \\ 
    Mean &0.5 &0.8852 &4.9684 &5.9038 &2.0227 \\
    Se &&0.0016 &0.0574 &0.0119 &0.0505 \\ 
    Mean &0.8 &0.8961 &4.7195 &5.8500 &1.8392 \\
    Se &&0.0013 &0.0633 &0.0146 &0.0385 \\ \hline
    \end{tabular*}
    \caption{Parameter recovery for the three parametric models, with \textbf{Weibull} baseline survival function. Simulations are based on the parameter values $(p_0,shape_0,scale_0)=(0.85,8,6)$ with 100 simulated samples of 100,000 families, with the true value of $\theta$ varying among (0.2, 0.5, 0.8).}
    \label{tab:1.4}
\end{table*}
\end{center}
% codice: simulation-par-contfraily-multivariate-MVV.R, simulation-par-contfrailty-univariate-MVV, simulation-par-univariateFh-MVV, simulation-par-contfrailty-multivariate-univariate-3pars-multidistribution-MVV.R    
\newpage
\begin{center}
\begin{table*}[!h]%
\begin{tabular*}{\textwidth}{@{\extracolsep\fill}llllll@{}}
\toprule
    &$\theta$ &$\widehat p_0$ &$\widehat shape_0$ &$\widehat scale_0$ &$\widehat\theta$ \\ \midrule
    Multivariate frailty Cure-Rate \\
    Mean &0.2 &0.8480 &8.2408 &5.9620 &\textbf{0.2252} \\
    Se &&1.1231 &1.0132 &0.0958 &0.0745 \\ 
    Mean &0.5 &0.8499 &8.0136 &5.9910 &\textbf{0.5001} \\
    Se &&0.0016 &0.0966 &0.0847 &0.0217 \\ 
    Mean &0.8 &0.8498 &7.9965 &6.0086 &\textbf{0.7985} \\
    Se &&0.0016 &0.0942 &0.0812 &0.0421 \\ \midrule
    Univariate frailty Cure-Rate \\
    Mean &0.2 &0.7897 &9.0371 &5.7925 &0.1283 \\
    Se &&0.0006 &0.0580 &0.0385 &0.0021 \\ 
    Mean &0.5 &0.7883 &8.4697 &6.0302 &0.1744 \\
    Se &&0.0009 &0.0384 &0.0335 &0.0039 \\ 
    Mean &0.8 &0.7762 &8.0837 &6.8830 &0.2169 \\
    Se &&0.0012 &0.0563 &0.0765 &0.0047 \\ \midrule
    &&&&&$\widehat\beta_F$ \\ \midrule
    Univariate FH Cure-Rate \\
    Mean &0.2 &0.6863 &4.5108 &53.5929 &2.5122 \\
    Se &&0.0103 &0.0776 &2.2848 &0.0662 \\ 
    Mean &0.5 &0.8278 &5.9820 &15.0471 &1.1020 \\
    Se &&0.0037 &0.0713 &0.6704 &0.0247 \\ 
    Mean &0.8 &0.8503 &6.7744 &8.9505 &0.7939 \\
    Se &&0.0012 &0.0737 &0.2033 &0.0247 \\ \bottomrule
    \end{tabular*}
    \caption{Parameter recovery for the three parametric models, with \textbf{Gamma} baseline survival function. Simulations are based on the parameter values $(p_0,shape_0,scale_0)=(0.85,8,6)$ with 100 simulated samples of 100,000 families, with the true value of $\theta$ varying among (0.2, 0.5, 0.8).}
    \label{tab:1.5}
\end{table*}
\end{center}
\newpage
\begin{center}
\begin{table*}[!h]%
\begin{tabular*}{\textwidth}{@{\extracolsep\fill}llllll@{}}
\toprule
    &$\theta$ &$\widehat{p}_0$ &$\widehat{\mu}_0$ &$\widehat{\sigma}_0$ &$\widehat\theta$ \\ \midrule
    Multivariate frailty Cure-Rate \\ 
    Mean &0.2 &0.8434 & 7.8914 & 5.9343 & \textbf{0.1975} \\ 
    Se && 0.0006 & 0.0192 & 0.0060 & 0.0004 \\ 
    Mean &0.5  & 0.8273 & 8.2779 & 6.0126 & \textbf{0.5495} \\ 
    Se & &0.0009 & 0.0238 & 0.0068 & 0.0017 \\ 
    Mean &0.8 & 0.8385 & 8.0476 & 6.0081 & 0.9485 \\ 
    Se & &0.0008 & 0.0227 & 0.0066 & 0.0039 \\  \midrule
    Univariate frailty Cure-Rate \\
    Mean &0.2 &0.8858 &6.0477 &5.4248 &0.3720 \\
    Se &&0.0001 &0.0015 &0.0015 &0.0036 \\ 
    Mean &0.5 &0.8872 &6.1642 &5.3674 &2.4196 \\
    Se &&0.0001 &0.0017 &0.0014 &0.0589 \\
    Mean &0.8 &0.8896 &6.0668 &5.3379 &0.6449 \\
    Se &&0.0001 &0.0028 &0.0021 &0.0052 \\ \midrule
    Univariate FH Cure-Rate &&&&&$\widehat\beta_F$ \\
    Mean &0.2 &0.9570 &0.3208 &3.7501 &0.1729 \\
    Se &&0.0004 &0.0465 &0.0472 & 0.0398 \\ 
    Mean &0.5 &0.9493 &0.8420 &3.8982 &-0.3137 \\
    Se &&0.0007 &0.0525 &0.0665 &0.0439 \\ 
    Mean &0.8 &0.9491 &1.0905 &3.6504 &-0.5317 \\
    Se &&0.0006 &0.0555 &0.0266 &0.0467 \\ \bottomrule
    \end{tabular*}
    \caption{Parameter recovery for the three parametric models, with \textbf{Lognormal} baseline survival function. Simulations are based on the parameter values $(p_0,\mu_0,\sigma_0)=(0.85,8,6)$ with 100 simulated samples of 100,000 families, with the true value of $\theta$ varying among (0.2, 0.5, 0.8).}
    \label{tab:1.6}
\end{table*}
\end{center}
\newpage
\begin{center}
\begin{table*}[!h]%
\begin{tabular*}{\textwidth}{@{\extracolsep\fill}lllllll@{}}
\toprule
    &$\theta$ &$\widehat p_0$ &$\widehat shape_0$ &$\widehat scale_0$ &$\widehat\gamma_0$ &$\widehat\theta$ \\ \midrule
    Multivariate frailty Cure-Rate \\ 
    Mean &0.2 &0.8408 & 9.3397 & 6.8492 & 13.4594 & \textbf{0.1956} \\ 
    Se &&0.0008 & 0.1751 & 0.0857 & 0.3991 & 0.0021 \\ 
    Mean &0.5 &0.8438 & 7.9902 & 7.6431 & 17.3172 & 0.6095 \\ 
    Se &&0.0004 & 0.1642 & 0.1154 & 0.3211 & 0.0121 \\ 
    Mean &0.8 &0.8651 &8.0636 &6.4814 &17.5597 &0.8833 \\
    Se &&0.0013 & 0.1817 & 0.2611 & 0.3511 & 0.0192 \\ \hline
    Univariate frailty Cure-Rate \\
    Mean &0.2 &0.7794 & 13.1800 & 4.5625 & 6.8431 & 102.8130 \\ 
    Se &&0.0069 & 0.1567 & 0.0368 & 0.3131 & 2.7683 \\ 
    Mean &0.5 &0.7648 & 13.1366 & 6.5679 & 9.1415 & 98.9017 \\ 
    Se &&0.0071 & 0.2535 & 0.1449 & 0.3799 & 2.6522 \\ 
    Mean &0.8 &0.7089 & 12.3406 & 7.7398 & 11.5973 & 68.9262 \\ 
    Se &&0.0079 & 0.2673 & 0.1807 & 0.4246 & 2.7657 \\  \hline
    Univariate FH Cure-Rate &&&&&&$\widehat\beta_F$ \\
    Mean &0.2 & 0.8991 & 8.6957 & 5.4870 & 14.3725 & 3.2352 \\ 
    Se &&$<$0.0001 & 0.0205 & 0.0084 & 0.0431 & 0.0048 \\ 
    Mean &0.5 & 0.8756 & 8.2294 & 5.9623 & 14.7768 & 1.6410 \\ 
    Se &&0.0001 & 0.0189 & 0.0110 & 0.0399 & 0.0022 \\ 
    Mean &0.8 &0.8729 & 6.8551 & 6.8521 & 18.3364 & 1.3828 \\ 
    Se &&0.0001 & 0.0208 & 0.0124 & 0.0413 & 0.0018 \\ \hline
    \end{tabular*}
    \caption{Parameter recovery for the three parametric models, with \textbf{3-parameters Gamma} baseline survival function. Simulations are based on the parameter values $(p_0,shape_0,scale_0,\gamma_0)=(0.85,8,6,15)$ with 100 simulated samples of 100,000 families.}
    \label{tab:1.7}
\end{table*}
\end{center}
% code in LOCAL: simulation-par-contfrailty-multivariate-univariate-3pars-gamma-MVV
\newpage 
\begin{table}
    \centering
    \begin{tabular}{l|cccccc}
    \hline
     Multivariate &$\theta$ &$\widehat p_0$ &$\widehat{\mu}_0$ &$\widehat{\sigma}_0$ &$\widehat\gamma_0$ &$\widehat\theta$ \\
    Mean &0.2 &0.4003 & 4.3076 & 9.8430 & 14.9639 & 1.1814 \\ 
    Se &&0.0006 & 0.0015 & 0.0055 & 0.0004 & 0.0019 \\ 
    Mean &0.5 & 0.4644 & 4.1581 & 9.2905 & 14.9893 & 1.1827 \\ 
    Se &&<0.0001 &<0.0001 &<0.0001 &<0.0001 &<0.0001 \\ 
    Mean &0.8 & 0.5811 & 3.3800 & 6.8228 & 14.9573 & 2.0971 \\ 
    Se &&0.0014 & 0.0037 & 0.0118 & 0.0003 & 0.0055 \\ \hline
    Univariate &$\theta$ &$\widehat p_0$ &$\widehat shape_0$ &$\widehat scale_0$ &$\widehat\gamma_0$ &$\widehat\theta$ \\
    Mean &0.2 & 0.3721 & 4.0631 & 9.3708 & 14.9097 & 2.5097 \\ 
    Se &&0.0012 & 0.0063 & 0.0147 & 0.0012 & 0.0173 \\ 
    Mean &0.5 &0.5613 & 3.4080 & 7.2050 & 14.9297 & 2.7473 \\ 
    Se &&0.0017 & 0.0046 & 0.0147 & 0.0010 & 0.0116 \\ 
    Mean &0.8 & 0.5885 & 3.4507 & 6.9386 & 14.9592 & 2.3564 \\ 
    Se &&0.0013 & 0.0046 & 0.0134 & 0.0004 & 0.0072 \\ \hline
    Univariate FH &$\theta$ &$\widehat p_0$ &$\widehat shape_0$ &$\widehat scale_0$ &$\widehat\gamma_0$ &$\widehat\beta_F$ \\
    Mean &0.2 & 0.9525 & 0.8947 & 3.1513 & 0.0911 & 0.1705 \\ 
    Se &&0.0007 & 0.0083 & 0.0081 & 0.0024 & 0.0045 \\  
    Mean &0.5 & 0.9358 & 0.8942 & 3.2681 & 0.1673 & 0.0895 \\ 
    Se &&0.0012 & 0.0060 & 0.0122 & 0.0051 & 0.0059 \\
    Mean &0.8 & 0.9302 & 0.9183 & 3.3025 & 0.1850 & 0.0618 \\ 
    Se &&0.0014 & 0.0062 & 0.0131 & 0.0059 & 0.0056 \\\hline
    \end{tabular}
    \caption{parameter recovery for the three parametric models, with \textbf{3-parameters Lognormal} baseline survival function. Simulations are based on the parameter values $(p_0,\mu_0,\sigma_0,\gamma_0)=(0.85,8,6,15)$ with 100 simulated samples of 100,000 families.}
    \label{tab:1.8}
\end{table}

\begin{figure}
    \centering
    \includegraphics[width = \linewidth]{PhD_thesis_template/plots/Rplot.pdf}
    \caption{Kaplan-Meier on main subjects with a Gamma baseline survival function for cases.}
    \label{fig:KM1}
\end{figure}
\begin{figure}
    \centering
    \includegraphics[width = \linewidth]{PhD_thesis_template/plots/Rplot01.pdf}
    \caption{Kaplan-Meier on main subjects with a Lognormal baseline survival function for cases.}
    \label{fig:KM2}
\end{figure}
\newpage
The findings presented in Tables \ref{tab:1.4}, \ref{tab:1.5}, \ref{tab:1.6}, \ref{tab:1.7}, and \ref{tab:1.8} align with our initial expectations, as the multivariate model was able to accurately identify and estimate the model parameters, given that the data was generated from a multivariate scenario, better than the univariate models. With the parameters identified, we can proceed to the risk prediction step by computing the posterior frailty mean, median, and mode using the parametric equations detailed in Section \ref{sec:1.4.2}. We still need to delve deeper into the analysis, explanation, and resolution of some identification issues, particularly regarding the three-parameters Lognormal distribution. 

Also, the cases Gamma distribution can almost accurately identify the true value of the parameters, used in the data generation process. Indeed, from the estimated Kaplan-Meier curve in Figure \ref{fig:KM1} we can appreciate how in these scenarios the survival curves remain constant before the time fixed as threshold, that in this case is set at 15, when censoring events and the event of interest begin to show up. At contrary, the cases Lognormal distribution shows some problems in correctly recovering the true values of the parameter. We can appreciate a particular behaviour of the estimated Kaplan-Meier curve in Figure \ref{fig:KM2}: the jump crossing the threshold is not smooth as in the Gamma scenario. We need to investigate more deeply about this issue. 
\begin{table}
    \centering
    \begin{tabular}{l|cccc} 
    \hline
    &\multicolumn{4}{c}{($n_f,\lambda_F$)} \\ 
    Multivariate &(2, 0.8) &(5, 0.8) &(10, 5) &(20, 10) \\ % &(50, 10) \\ 
MSPE(mean) &3.873(0.049) &4.800(0.074) &3.843(0.071) &\hl{\textbf{3.433(0.049)}} \\ % &4.084(0.053) \\ 
MSPE(median) &4.420(0.052) &5.401(0.078) &4.177(0.074) &\hl{\textbf{3.643(0.051)}} \\ % &4.319(0.055) \\ 
MSPE(mode) &11.348(0.061)&12.198(0.088)&6.009(0.082) &\hl{\textbf{4.430(0.052)}} \\ % &5.168(0.059) \\ 
$R^2$(mean) &0.202(0.002) &0.219(0.002)	&0.478(0.004) &\hl{\textbf{0.614(0.002)}}  \\ % &\hl{\textbf{0.624(0.003)}} \\ 
$R^2$(median) &0.207(0.002) &0.222(0.002) &0.485(0.004) &\hl{\textbf{0.619(0.002)}} \\ % &\hl{\textbf{0.624(0.003)}} \\ 
$R^2$(mode) &0.127(0.002) &0.115(0.002)	&0.383(0.003) &\hl{\textbf{0.583(0.002)}} \\ % &0.581(0.002) \\ 
$\widehat{\beta}_{\text{mean}}$ &\hl{\textbf{1.553(0.008)}} &1.705(0.011) &2.401(0.018)	&2.781(0.018) \\ % &2.948(0.008) \\ 
$\widehat{\beta}_{\text{median}}$ &\hl{\textbf{1.792(0.010)}} &1.975(0.013)	&2.608(0.019) &2.932(0.019) \\ % &3.099(0.009) \\ 
$\widehat{\beta}_{\text{mode}}$ &\hl{\textbf{0.957(0.007)}} &0.927(0.009) &1.929(0.015)	&2.645(0.017) \\ % &2.767(0.009) \\ 
$\rho$(mean) &0.447(0.002) &0.465(0.002) &0.689(0.007) &\hl{\textbf{0.783(0.001)}} \\ % &\hl{\textbf{0.787(0.002)}} \\ 
$\rho$(median) &0.452(0.002) &0.468(0.002) &0.694(0.003) &\hl{\textbf{0.786(0.001)}} \\ % &\hl{\textbf{0.789(0.002)}} \\ 
$\rho$(mode) &0.353(0.002) &0.335(0.003) &0.616(0.002) &\hl{\textbf{0.763(0.001)}} \\ % &0.761(0.002) \\ 
Rank $\rho$(mean) &0.181(0.002) &0.226(0.002) &0.438(0.002) &\hl{\textbf{0.539(0.001)}} \\ % &\hl{\textbf{0.559(0.002)}} \\ 
Rank $\rho$(median) &0.181(0.002) &0.226(0.002) &0.438(0.002) &\hl{\textbf{0.539(0.001)}} \\ % &\hl{\textbf{0.559(0.002)}} \\ 
Rank $\rho$(mode) &0.151(0.002) &0.146(0.002) &0.332(0.002) &\hl{\textbf{0.490(0.002)}} \\ % &\hl{\textbf{0.492(0.002)}} \\ 
\hline
Univariate \\
MSPE(mean) &\textbf{4.241(0.053)} &5.259(0.081)	&4.738(0.084) &4.507(0.059) \\ % &5.258(0.063) \\ 
MSPE(median) &\textbf{4.536(0.055)} &5.594(0.084) &5.040(0.088)	&4.798(0.062) \\ % &5.598(0.067) \\ 
MSPE(mode) &\textbf{6.511(0.060)} &7.666(0.091)	&7.019(0.098) &6.767(0.068) \\ % &7.685(0.076) \\ 
$R^2$(mean) &0.136(0.003) &0.157(0.003)	&\textbf{0.183(0.002)} &0.126(0.002) \\ % &0.153(0.002) \\ 
$R^2$(median) &0.138(0.003) &0.161(0.003)	&\textbf{0.191(0.003)}	&0.131(0.002) \\ % &0.157(0.002) \\ 
$R^2$(mode) &0.128(0.003) &0.149(0.002)	&\textbf{0.190(0.002)} &0.128(0.002) \\ % &0.147(0.002) \\ 
$\widehat{\beta}_{\text{mean}}$ &2.704(0.025) &3.158(0.021)	&3.326(0.037) &\textbf{2.685(0.023)} \\ % &3.175(0.027) \\ 
$\widehat{\beta}_{\text{median}}$ &2.893(0.028) &3.369(0.023) &3.588(0.038) &\textbf{2.887(0.025)} \\ % &3.393(0.028) \\ 
$\widehat{\beta}_{\text{mode}}$ &\textbf{2.621(0.028)} &3.032(0.022) &3.349(0.035) &2.658(0.022) \\ % &3.055(0.024) \\ 
$\rho$(mean) &0.356(0.004) &0.391(0.003)	&\textbf{0.422(0.003)} &0.351(0.003) \\ % &0.387(0.003) \\ 
$\rho$(median) &0.359(0.004) &0.395(0.003)	&\textbf{0.432(0.003)} &0.357(0.003) \\ % &0.392(0.003) \\ 
$\rho$(mode) &0.348(0.004) &0.381(0.003)	&\textbf{0.431(0.003)} &0.353(0.002) \\ % &0.379(0.002) \\ 
 Rank $\rho$(mean) &0.201(0.004) &\textbf{0.206(0.002)} &0.194(0.002) &0.183(0.002) \\ % &\textbf{0.210(0.002)} \\ 
 Rank $\rho$(median) &0.201(0.004) &\textbf{0.206(0.002)}	&0.194(0.002) &0.183(0.002) \\ % &\textbf{0.210(0.002)} \\ 
 Rank $\rho$(mode) &0.167(0.003) &0.190(0.002) &\textbf{0.203(0.002)} &0.192(0.001) \\ % &0.175(0.001) \\ 
 \hline
Univariate FH \\
MSPE &5.388(0.057) &6.573(0.089) &5.625(0.097) &\textbf{5.034(0.062)} \\ % &5.988(0.069) \\ 
$R^2$ &0.043(0.001) &0.043(0.001) &0.152(0.003) &\textbf{0.186(0.002)} \\ % &0.152(0.002) \\ 
$\widehat{\beta}_{\text{fh}}$ &3.584(0.077) &3.971(0.077) &3.222(0.039)	&\textbf{2.806(0.019)} \\ % &2.828(0.023) \\ 
$\rho$ &0.189(0.004) &0.195(0.003) &0.379(0.004) &\textbf{0.429(0.002)} \\ % &0.388(0.002) \\ 
Rank $\rho$(mean) &0.147(0.002) &0.159(0.002) &0.348(0.001)	&\textbf{0.435(0.001)} \\ % &0.426(0.001) \\ 
\hline
    \end{tabular}
    \caption{several prediction accuracy measures, for different family sizes with values of ($n_F, \lambda_F$). All the results are reported in average value over the 100 iterations and standard error in brackets.}
    \label{tab:1.9}
\end{table}
\newline

Once the parameter recovery is assessed for, above all, the multivariate model, we can move to the analysis of the posterior risk frailty from the posterior frailty distribution. We report for the family size parameters varying at $(n_f,\lambda_F)=((2,0.8),(5,0.8),(10,5),(20,10)^T$, the mean squared prediction error, the coefficient of determination, the coefficient of the predicted frailty covariate, the correlation coefficient, and the rank correlation coefficient. In the multivariate and univariate frailty scenario the predicted frailty is identified with the mean, the median, or the mode from the posterior frailty distribution. While in the univariate model with $FH$ the ``estimated frailty'' is the family history indicator itself. In bold are the best results by row, and in yellow are the best results by category across all the models. 

From Table \ref{tab:1.9} we can claim the following assessments: the mean square prediction error is not getting lower and lower as the family size increases, on the contrary of what we can expect. This can be due to the random generation of the samples every time. However, for the posterior mode frailty we can notice that with the increasing of the family size the mean square prediction error is halved. By comparing the multivariate model, the univariate model and the univariate FH model we can not notice a significative difference in terms of MSPE, with the best results by the multivariate model. The coefficient of determination is instead increasing when the family size tends to a bigger number, This is exactly what we expect from before the analysis: a sample with bigger families has a greater explainability power on the breast cancer development than a sample with families composed by few subjects. Moreover, here we can appreciate the higher power of the multivariate model in comparison to the other two because of the multivariate nature of itself. In the univariate models, the coefficient of determination is always computed on one subject per family. The best result is when the multivariate model with mean or median frailty as covariate, explains the 61.4\% of variability of the true frailty with the maximum values of family parameters. For what concerns the parameter of the frailty into the regression true frailty vs. predicted frailty, we would like a value that is as closest as possible to one. Here we notice that this value is increasing with the family size for the multivariate model, but for the univariate models we notice that it is around three independently to the parameters of the family size. This coefficient is not that meaningful. The correlation of Pearson and the rank correlation increase when the family size increases, as we expect before the analysis. Moreover, in the multivariate model these indices are higher than into the univariate models, but few rare cases. The best results are a Pearson correlation of 0.789 between the true frailty and the median frailty estimated through the multivariate likelihood, and the rank correlation with a value of 0.559 between the true frailty and both the mean and the median frailty, always estimated through the multivariate likelihood. Once again, the multivariate model outperforms the other, as expected and hoped.  
% you can apply C:/Users/Vinattieri/OneDrive - Universit Commerciale Luigi Bocconi/Research/Familiarity/simulated data/simulation-par-contfrailty-multiUniFh-postprediction-MVV.R in LOCAL COMPUTER only for the part of mspe...
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% \begin{table}[ht]
% \centering
% \begin{tabular}{rrr}
%   \hline
%  & ave & se \\ 
%   \hline
% CONCORDANCE\_multi & 0.9628 & 0.0000 \\ 
%   CONCORDANCE\_fh & 0.9250 & 0.0000 \\ 
%   CONCORDANCE\_uni & 1.0000 & 0.0000 \\ 
%   AUC & 0.6496 & 0.0001 \\ 
%   X & 0.5741 & 0.0001 \\ 
%   X.1 & 0.6288 & 0.0001 \\ 
%   PPV & 0.2642 & 0.0002 \\ 
%   X.2 & 0.1497 & 0.0001 \\ 
%   X.3 & 0.2022 & 0.0002 \\ 
%   NPV & 0.9613 & 0.0000 \\ 
%   X.4 & 0.9304 & 0.0000 \\ 
%   X.5 & 0.9580 & 0.0000 \\ 
%   MSPE\_multi\_mean & 1.7798 & 0.0007 \\ 
%   MSPE\_multi\_median & 1.6494 & 0.0005 \\ 
%   MSPE\_multi\_mode & 7.1650 & 0.0006 \\ 
%   MSPE\_fh & 2.0108 & 0.0005 \\ 
%   MSPE\_uni\_mean & 1.1015 & 0.0004 \\ 
%   MSPE\_uni\_median & 1.2816 & 0.0004 \\ 
%   MSPE\_uni\_mode & 3.0871 & 0.0006 \\ 
%   R2 & 0.2014 & 0.0001 \\ 
%   X.6 & 0.2014 & 0.0001 \\ 
%   X.7 & 0.1508 & 0.0001 \\ 
%   X.8 & 0.0525 & 0.0001 \\ 
%   X.9 & 0.1331 & 0.0001 \\ 
%   X.10 & 0.1335 & 0.0001 \\ 
%   X.11 & 0.1292 & 0.0001 \\ 
%   COEFF.pMeanMulti & 0.3587 & 0.0002 \\ 
%   pMedianMulti & 0.4156 & 0.0002 \\ 
%   pModeMulti & 0.3330 & 0.0001 \\ 
%   fh & 0.9468 & 0.0007 \\ 
%   pMeanUni & 0.6939 & 0.0003 \\ 
%   pMedianUni & 0.7586 & 0.0003 \\ 
%   pModeUni & 0.8076 & 0.0004 \\ 
%   CORR & 0.4486 & 0.0001 \\ 
%   X.12 & 0.4486 & 0.0001 \\ 
%   X.13 & 0.3881 & 0.0001 \\ 
%   X.14 & 0.2287 & 0.0002 \\ 
%   X.15 & 0.3646 & 0.0001 \\ 
%   X.16 & 0.3652 & 0.0001 \\ 
%   X.17 & 0.3591 & 0.0001 \\ 
%   RANKCORR & 0.3542 & 0.0001 \\ 
%   X.18 & 0.3544 & 0.0001 \\ 
%   X.19 & 0.2412 & 0.0001 \\ 
%   X.20 & 0.2272 & 0.0001 \\ 
%   X.21 & 0.2870 & 0.0001 \\ 
%   X.22 & 0.2870 & 0.0001 \\ 
%   X.23 & 0.2623 & 0.0001 \\ 
%    \hline
% \end{tabular}
% \end{table}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

Also, we can compare the models by applying the binary splitting of the predicted frailty and reporting the AUC, the PPV, the NPV and the Harrell's concordance index C in Table \ref{tab:1.10}. The multivariate outperforms the other two univariate models, with an AUC in the range 70\%-95\%, increasing according to a higher family size, against an AUC around the 62\% for the univariate frailty, and in the range 50\%-70\%, more or less, for the univariate $FH$ model. These additional 20 percentile points in the multivariate model allows to reach a strong predictive power that is unmatched. Similarly to the semiparametric scenario, also here the positive predictive value is weaker than the negative predictive power, meaning that the low-risk families are better identified than the highest-risk families. This allows for a reduction of unnecessary costs, medical treatments and psychological stress for families who do not need to be under surveillance. 
\begin{table}[ht]
    \centering
    \begin{tabular}{l|cccc} 
    \hline
    &\multicolumn{4}{c}{($n_F, \lambda_F$)} \\ 
    Multivariate &(2, 0.8) &(5, 0.8) &(10, 5) &(20, 10) \\ % &(50, 10) \\
AUC &0.697(0.002) &0.714(0.002)	&0.891(0.002) &\hl{\textbf{0.948(0.001)}} \\ % &\hl{\textbf{0.953(0.001)}} \\ 
PPV &0.318(0.002) &0.352(0.003)	&0.532(0.003) &\hl{\textbf{0.624(0.003)}} \\ % &\hl{\textbf{0.589(0.003)}} \\ 
NPV &0.951(<0.001) &0.959(<0.001)	&0.975(<0.001) &\hl{\textbf{0.980(<0.001)}} \\ % &0.978(<0.001) \\ 
C &\hl{\textbf{0.960(<0.001)}} &0.959(<0.001) &0.943(0.001)	&0.929(0.001) \\ % &0.929(0.001) \\ 
\hline 
Univariate \\
AUC &0.627(0.002) &\textbf{0.637(0.002)	}&0.632(0.002) &0.621(0.001) \\ % &\textbf{0.639(0.001)} \\ 
PPV &0.286(0.004) &0.332(0.003)	&\textbf{0.373(0.004)}	&0.332(0.003) \\ % &0.319(0.003) \\ 
NPV &0.959(<0.001) &0.961(<0.001)	&\textbf{0.965(<0.001)}	&0.963(<0.001) \\ % &0.962(<0.001) \\ 
C &\textbf{0.952(<0.001)} &0.947(<0.001)	&0.951(<0.001)	&0.950(<0.001) \\ % &0.949(<0.001) \\ 
\hline
Univariate FH \\ 
AUC &0.535(0.001) &0.533(0.001)	&0.645(0.002)	&\textbf{0.712(0.001)} \\ % &0.690(0.001) \\ 
PPV &\textbf{0.314(0.009)} &0.299(0.009)	&0.288(0.003)	&0.223(0.001) \\ % &0.202(0.002) \\ 
NPV &\textbf{0.991(<0.001)} &0.989(<0.001)	&0.944(<0.001)	&0.897(<0.001) \\ % &0.899(0.001) \\ 
C &-&-&-&- \\ \hline
% C &\textbf{0.980(<0.001)} &0.976(<0.001)	&0.918(<0.001)	&0.908(0.001) &0.895 (0.001)\\ 
\end{tabular}
\caption{average AUC, PPV, NPV, and Harrell's concordance index, with standard errors, corresponding to the ROC curve on the binary highest-risk group indicator with threshold $1 - \alpha = 0.95$. Family size is varying with ($n_F, \lambda_F$).}
    \label{tab:1.10}
\end{table}

We refer to Figure \ref{fig:1.8} for an example of ROC curve where the mean frailty from the multivariate model has been taken as frailty prediction. The threshold is shifting automatically in the plot. We can appreciate how the AUC is significantly increasing as the family size grows in the sample from the 88\% with a maximum of five members, to 95\% with a maximum of twenty members per family.
\begin{figure}
    \centering
    \includegraphics[width=\linewidth]{plots/Rplot08.pdf}
    \includegraphics[width=\linewidth]{plots/Rplot11.pdf}
    \caption{ROC curve on 100,000 families of the mean frailty risk versus the true membership to the group of the highest-risk families (top 5\%). Family size is varying with the values of ($n_f, \lambda_F$): (5, 0.8), (20, 10).}
    \label{fig:1.8}
\end{figure}

% code is in C:/Users/Vinattieri/OneDrive - Universit Commerciale Luigi Bocconi/Research/Familiarity/simulated data/simulation-par-contfrailty-multiUniFh-postprediction-MVV.R in LOCAL COMPUTER only for the part of auc, ppv, npv and concordance. 
% code for plot LOCAL: postPredictionPlots.R
Now, we can conclude by stating that the estimation of frailty mean through the maximization of the multivariate likelihood yields the most favorable results. Furthermore, the larger the family size in the sample, the better the performance of this index. Please refer to Figure \ref{fig:1.7} to observe the improved accuracy in prediction as the family size increases along with the mean frailty. The observed behavior in the plot with smaller family parameters is attributed to the aforementioned shrinkage issue of the mean frailty, which is effectively mitigated with a larger family size. 
\begin{figure}
    \centering
    \includegraphics[width=\linewidth]{plots/Rplot06.pdf}
    \includegraphics[width=\linewidth]{plots/Rplot09.pdf}
    \caption{scatter-plot on 100,000 families of true frailty value versus predicted mean frailty with the bisector of the third quadrant (in red), the smoothed regression line (in green with confidence interval) and smoothed quantile regression line on the first, second and third quantile (in orange).}
    \label{fig:1.7}
\end{figure}

No distinction can be observed in the conditional MSPE (mean squared prediction error), plotted against the true frailty value for 100,000 families in Figure \ref{fig:1.9}, between small family sizes and large family sizes. This observation highlights the need to explore alternative estimations for frailty risk in order to overcome the various challenges encountered with the mean estimation.
\begin{figure}
    \centering
    \includegraphics[width=\linewidth]{plots/Rplot07.pdf}
    \includegraphics[width=\linewidth]{plots/Rplot10.pdf}
    \caption{conditional MSPE for family size varying with the values $(n_F, \lambda_F)$: (5, 0.8), (20, 10).}
    \label{fig:1.9}
\end{figure}

In conclusion, the multivariate model has the highest predictive power also for the binary splitting in terms of area under the ROC curve, reaching the 95.3\% of accuracy in prediction, a positive predictive value, with the highest value of 58.9\% of families truly belonging to the highest-risk group. Moreover, the multivariate model gives the best result in terms of negative predictive power, with the 98\% of families truly not belonging to the highest-risk group. In the multivariate model, 96\% of the pairs exhibit concordance between the observed time of diagnosis and risk values. In comparison, the univariate model shows a slightly lower concordance rate of up to 95.2\%. It is worth noting that for the univariate model with the $FH$ variable, the concordance index cannot be calculated without adjusting for ties. This is due to the presence of ties caused by the $FH$ indicator, which trivially can only have values of 0 or 1.
\clearpage 

\section[Illustration to the Swedish Breast Cancer registry]{Illustration to the Swedish Multi-Generational Breast Cancer registry}\label{sec:1.6}
\subsection{Motivating framework}
Cancer is a prevalent global health concern, and breast cancer stands as the most prevalent form of cancer among females, comprising approximately 30$\%$ of all cancer diagnoses \cite{siegel2019cancer}. The presence of lesions in secondary locations is a significant risk factor for cancer, as these lesions are known to be highly fatal due to the process of metastasis. Metastasis is responsible for over 90$\%$ of cancer-related deaths \cite{riggio2021lingering}.

In recent decades, a lot has  been discovered in cancer research with the goal of improving the survival rates of patients. Despite these progresses, a significant amount of information regarding the development of cancer remains unknown, which presents an opportunity for further improvement. Specifically, areas in which there is potential for improvement include the detection and treatment of primary tumors and metastases. This section will provide an overview of the most recent literature on the etiology, diagnosis, and treatment of cancer. Of particular interest is the advancement of early detection methods, as early detection is crucial for increasing treatment options and the probability of a successful recovery.

Primary cancers have three distinct causes, namely genetic factors, individual lifestyle, and external environmental factors. In the context of breast cancer, several genes have been identified as crucial for its development. These include breast cancer genes (BRCA1, BRCA2), which are associated with the highest incidence of breast and ovarian cancers in families, as well as the tumor protein mutation (TP53), the single nucleotide polymorphisms (SNPs), and the polygenic risk scores (PRS) \citep{riskfactor, loibl2021breast}. BRCA1 and BRCA2 mutations are highly indicative of cancer detection; however, they only account for $20-25\%$ of familial cancer aggregation \cite{riggio2021lingering}. As a result, the remaining $75\%$ of cancer familial aggregation is unknown.

Other risk factors for breast cancer, not genetic, include mammography density (MD), age, oral contraceptives, parity and timing of births, breastfeeding, age at menarche and menopause, body mass index (BMI), physical exercise, alcohol and tobacco consumption, and family history \cite{eriksson2021risk}. Among these, family history is considered one of the strongest. The term family history refers to the incidence of breast cancer onset among family members. A positive family history is characterized by at least one family member that has experienced breast cancer onset at the time of the analysis. Furthermore, the magnitude of the family history effect depends on the number and grade of relatives who have experienced the disease. For example, a positive family history in first-degree relatives, such as the mother or sisters, has a greater impact compared to breast cancer cases in second or third-degree relatives, such as grandmothers and aunts \citep{colditz1996risk}.

Now that we have analysed the possible causes, we explore the three-step process of breast cancer diagnosis. The first step is the clinical analysis, which is the least invasive. If necessary, the process proceeds to the intermediate step of machine imaging. Digital mammography is the most commonly used imaging technique. The final and most invasive step involves a biopsy, where a fine needle is used for cytopathology diagnosis, which studies pathology at the cellular level. After the identification of a tumor, it has to be classified into one of the categories \cite{eriksson2021risk}.

Tumors can be categorized using different methods, such as primary origin and metastasis stage, histological grade, or molecular subtypes, which are determined by the phenotype, namely the visible expression, of the cells. There are five molecular subtypes of invasive breast cancer: luminal A, luminal B, luminal B-like, HER2-enriched, and triple negative. These categories are mainly determined by the expression of three receptors: estrogen receptor (ER), progesterone receptor (PR), and human epidermal growth factor receptor type 2 (HER2). Luminal cancers are the least dangerous as they need estrogen to grow, thus blocking the estrogen receptor is often sufficient to stop tumor growth. Luminal A is ER positive, PR positive, and HER2 negative, while luminal B cancer is ER positive and HER2 negative, with any result on PR. Luminal B-like cancer is ER positive, PR negative, and HER2 positive. The HER2-enriched is ER and PR negative, but HER2 positive. The triple negative (TN) tumor has ER, PR, and HER2 negative, and is often referred to as TN-negative and it is the most dangerous for patient's survival. These categories are also grouped as HR-positive for luminal tumors, HER2-positive, and TN-negative.

Once that the breast cancer has been categorized, the  treatment can be carried out according the the category. 

Luminal A cancer, which grows slowly and has a better prognosis, can be treated with hormonal therapy, which is the most basic and least invasive therapy. Luminal B and luminal B-like cancers grow faster and have a worse prognosis, but they can typically be treated with appropriate hormonal therapy. HER2-enriched cancer is worse than luminal tumors, and it may require chemotherapy  and targeted immunotherapy as primary treatments, followed by hormonal therapy. Triple negative cancer is the most dangerous subtype, often occurring in individuals with a BRCA1 mutation, young people, and black people. It can be managed through chemotherapy and adjuvant chemotherapy (see e.g. \cite{gelber1996adjuvant}), with surgery as a subsequent option if necessary. The research on the treatment of breast cancer based on molecular subtypes is currently the most advanced.

Breast cancer is a complex disease that can metastasize, or spread, to other parts of the body. According to recent studies, approximately $30\%$ of breast cancer patients develop metastasis \cite{metastisperc}. Therefore, understanding metastasis is crucial for improving patient survival outcomes. Metastasis is a process where cancer cells escape from the primary tumor, which is the original site of cancer, and disseminate to a secondary location. Metastasis can be categorized as regional or distant, depending on the distance the cancer cells have travelled. Regional metastasis occurs when the cancer cells invade the regional lymph nodes. Distant metastasis occurs when cancer cells enter the bloodstream or lymphatic vessels and disseminate to other organs in the body. The cancer cells must survive in circulation to proceed with metastasis, and then invade other organs, leading to the seeding process. The most common organs for distant metastasis are skeleton, lungs, liver, and brain, with the brain being the least frequent but the most harmful \cite{strandberg2022breast}.

It is still unclear when the metastasis process begins. Researchers have proposed two models of metastatic dissemination: the linear model and the parallel model \cite{riggio2021lingering}. The linear model suggests that the metastatic progression occurs after the primary tumor grows and becomes malignant. With the linear dissemination model, metastasis occurs only later in the tumor cell development. On the other hand, the parallel model suggests that metastatic progression happens simultaneously with primary tumor development. This hypothesis is supported by several phenomena, such as early-stage cancer patients found with secondary tumors and second cancer lesions discovered in patients with cancer of unknown origin. The parallel dissemination is much more dangerous than the linear progression. Notice that, both the linear and parallel dissemination can occur within the same patient. 

Also the treatment of metastasis depends on the molecular subtype of breast cancer. Currently, metastatic breast cancer is considered an incurable disease, and the focus is on prolonging patient survival and improving their quality of life. The treatment options include systemic therapies, such as chemotherapy, hormonal therapy, and targeted therapy. Surgical intervention and radiation therapy may also be used in specific cases. Regular monitoring and imaging tests are necessary to monitor the progression of metastatic breast cancer and adjust the treatment plan accordingly. 

After having explored all the details characterizing breast cancer, we can now turn the attention to our main purpose of analysis. As previously mentioned, a significant proportion, namely the $75\%$, of familial aggregation of cancer remains unexplained. It is our goal to investigate this phenomenon and contribute to increase the knowledge about the hereditary aspect of cancer. Most importantly, let us highlight that both the diagnosis and prognosis of breast cancer are influenced by familial factors. Indeed, individuals with breast cancer cases among first-degree relatives have twice the risk of developing breast cancer compared to individuals from families with no such cases \cite{antoniou2001evidence}.

Therefore, our focus is to investigate the role of a latent component $R$ in explaining the unexplained familial aggregation of breast cancer. We assume that the families can be characterized by $R$, which is the true genetic risk of developing breast cancer that remains constant from birth. Our aim is to perform risk classification or prediction according to the discrete or continuous nature of the risk. 

Accurately classifying women into the correct risk group has numerous benefits, some of which are mentioned below \citep{pepe2003statistical}: firstly, it enables families to be targeted towards different prevention paths. Secondly, it helps in modifying screening schedules according to the individual risk of developing breast cancer, which can either be more or less intensive. Thirdly, it facilitates the implementation of tailored preventive strategies for patients and raises awareness regarding the risk of developing breast cancer, leading to better medical surveillance and preventive care \citep{evans1996fictitious}. Lastly, it leads to a reduction of unnecessary medical treatments, costs, and psychological stress for women who do not require treatment. 

\subsection{The Data}
Sweden's centralized information storage process enables the creation of comprehensive and easily accessible registries. Notably, all the information is linked to the patient's unique identification number, known as the ``lpnummer'' in Swedish. This allows the demographic and medical information to be traced back to each patient and family. The primary focus of this study is on breast cancer data, death, and migration flux, which are available in separate dataset. By merging these dataset, comprehensive information on each woman within each family can be obtained. These exceptional registries not only allow us to identify the family members but also link them together. Specifically, the Multi-Generational Registry provides information on the relationships between family members. The process of constructing the clean dataset is detailed in Appendix \ref{appendix:1.d}.

The dataset concerns a cohort of $n = 1,603,920$ Swedish families, consisting of a total of 4,267,803 women.
% code in ParContinuousFrailty-multidistribution-multivariate-swemultireg-MVV SWEDEN
Among these families, $1,603,920$ women (one per family) were born between the 1947 January 1st, and 1976 December 31st, so that they age between 40 and 70 years old at the end of the follow-up (which coincides with 2016, December 30th), and they are what we call the ``main subject'' in the family, which are also the women included in the dataset as observations (on the rows), and not only as relatives of a main subject (on the columns). This age window has been chosen as it corresponds to the time when women have the highest chances to develop breast cancer. The reason why we identify a main subject in each family is for moving to the univariate counterpart of the models where each family is composed by only one subject. The other family components are the mother and the sisters of the main subject. The number of sisters can vary between zero and twelve. The main subject is chosen as follows: initially, one daughter is randomly selected for each mother. Subsequently, one between the mother and that one daughter is randomly chosen. This selection process ensures that the mother is chosen with a probability of $1/2$. If we randomly sample one subject per family without this two-steps selection process, the probability of selecting the mother would not reach 1/2 when there are two or more sisters in the family.

For each main subject and each relative of the main subject we need to obtain complete information. Thus, we move to merge the Multi-Generational Registry, the Swedish Cancer Registry, the Cause of Death Registry, and the Migration Registry together. Notice that we use these last two registries solely for building the censoring time variable. We select only those variables relevant to our problem and filter on the female subjects since the breast cancer incidence among males is negligible. Our analysis is based on the two-generations family history of breast cancer, i.e., to the value of the family history, computed on the main subject, contribute her mother and her sisters. We exclude the non-invasive DCIS (ductual carcinoma in-situ) diagnose as event of interest. Competing events to breast cancer diagnosis (which we treat as disease ``onset'') are death or emigration. The Death Registry provides information on the death date, while the Migration Registry provides the time of the last emigration, assuming no breast cancer onset occurred in other countries if the woman immigrated back to Sweden as her last movement, as we lack the information to confirm otherwise. 

All our analyses were conducted following the approval by the Bocconi Ethics Board.

\subsection{A preliminary Case-Control analysis} % qui 
We start by implementing a Nested Case-Control (NCC) study design to estimate the relative risk (RR) of developing breast cancer for women with a positive first-degree family history (mother or sisters) compared to those without a family history. We aim to obtain a value similar to the well-known estimated value from literature, which is RR = 1.8 \cite{collaborative2001familial}. To conduct the NCC study, we select subjects based on specific criteria from our initial dataset of 6,633,147 subjects. First, we select women with a birthday within the end of the follow-up and with their mother in the dataset, resulting in 3,798,079 subjects. We further narrow the dataset to women aged between 40 and 70 years old in the time window of 2010-2016, resulting in 1,131,499 subjects. Thus, we match subjects based on their year of birth and select 19,550 cases and 90,833 unique controls (five controls for each case) through a process of sampling with reintroduction. This results in a final dataset of 114,330 subjects, which we call the ``Nested Case-Control dataset'' (NCCD), whose consort plot \cite{moher2001consort} is in Figure \ref{fig:nccd} with all the passages from the initial dataset to the one obtained for a proper analysis. 

We carry out a survival Cox model with the time-varying family history covariate $FH(t)$ on the NCCD \cite{zhang2018time}, and the exponential of the estimated coefficient of family history is 1.8017, which is aligned with our expected value 1.8 \cite{collaborative2001familial, pharoah1997family}. This result validates the quality of our data. 
\begin{figure}[ht]
\centering
\tikzset{every picture/.style={line width=0.5pt}} %set default line width to 0.75pt     
\begin{tikzpicture}[x=0.75pt,y=0.75pt,yscale=-0.8,xscale=0.8]
%uncomment if require: \path (0,327); %set diagram left start at 0, and has height of 327

%Straight Lines [id:da09755630118978076] 
\draw    (252,252) -- (292.15,288.41) ;
\draw [shift={(293.63,289.75)}, rotate = 222.2] [color={rgb, 255:red, 0; green, 0; blue, 0 }  ][line width=0.75]    (10.93,-3.29) .. controls (6.95,-1.4) and (3.31,-0.3) .. (0,0) .. controls (3.31,0.3) and (6.95,1.4) .. (10.93,3.29)   ;
%Straight Lines [id:da44638779412348717] 
\draw    (152,250) -- (112.08,288.36) ;
\draw [shift={(110.63,289.75)}, rotate = 316.14] [color={rgb, 255:red, 0; green, 0; blue, 0 }  ][line width=0.75]    (10.93,-3.29) .. controls (6.95,-1.4) and (3.31,-0.3) .. (0,0) .. controls (3.31,0.3) and (6.95,1.4) .. (10.93,3.29)   ;
%Straight Lines [id:da8871200673466652] 
\draw    (203,141) -- (203.59,168.75) ;
\draw [shift={(203.63,170.75)}, rotate = 268.78] [color={rgb, 255:red, 0; green, 0; blue, 0 }  ][line width=0.75]    (10.93,-3.29) .. controls (6.95,-1.4) and (3.31,-0.3) .. (0,0) .. controls (3.31,0.3) and (6.95,1.4) .. (10.93,3.29)   ;
%Straight Lines [id:da1648623506457928] 
\draw    (203.32,155.88) -- (441.63,155.75) ;
\draw [shift={(443.63,155.75)}, rotate = 179.97] [color={rgb, 255:red, 0; green, 0; blue, 0 }  ][line width=0.75]    (10.93,-3.29) .. controls (6.95,-1.4) and (3.31,-0.3) .. (0,0) .. controls (3.31,0.3) and (6.95,1.4) .. (10.93,3.29)   ;
%Straight Lines [id:da768775166366618] 
\draw    (204,192) -- (204.59,219.75) ;
\draw [shift={(204.63,221.75)}, rotate = 268.78] [color={rgb, 255:red, 0; green, 0; blue, 0 }  ][line width=0.75]    (10.93,-3.29) .. controls (6.95,-1.4) and (3.31,-0.3) .. (0,0) .. controls (3.31,0.3) and (6.95,1.4) .. (10.93,3.29)   ;
%Straight Lines [id:da9792209051883168] 
\draw    (204.32,200.88) -- (442.63,200.75) ;
\draw [shift={(444.63,200.75)}, rotate = 179.97] [color={rgb, 255:red, 0; green, 0; blue, 0 }  ][line width=0.75]    (10.93,-3.29) .. controls (6.95,-1.4) and (3.31,-0.3) .. (0,0) .. controls (3.31,0.3) and (6.95,1.4) .. (10.93,3.29)   ;
%Straight Lines [id:da7032136969785114] 
\draw    (202,89) -- (202.59,116.75) ;
\draw [shift={(202.63,118.75)}, rotate = 268.78] [color={rgb, 255:red, 0; green, 0; blue, 0 }  ][line width=0.75]    (10.93,-3.29) .. controls (6.95,-1.4) and (3.31,-0.3) .. (0,0) .. controls (3.31,0.3) and (6.95,1.4) .. (10.93,3.29)   ;
%Straight Lines [id:da2824593510407103] 
\draw    (202.32,45.88) -- (434.63,44.76) ;
\draw [shift={(436.63,44.75)}, rotate = 179.72] [color={rgb, 255:red, 0; green, 0; blue, 0 }  ][line width=0.75]    (10.93,-3.29) .. controls (6.95,-1.4) and (3.31,-0.3) .. (0,0) .. controls (3.31,0.3) and (6.95,1.4) .. (10.93,3.29)   ;
%Straight Lines [id:da6294357350000164] 
\draw    (202,31) -- (202.59,58.75) ;
\draw [shift={(202.63,60.75)}, rotate = 268.78] [color={rgb, 255:red, 0; green, 0; blue, 0 }  ][line width=0.75]    (10.93,-3.29) .. controls (6.95,-1.4) and (3.31,-0.3) .. (0,0) .. controls (3.31,0.3) and (6.95,1.4) .. (10.93,3.29)   ;
%Straight Lines [id:da5602976120653284] 
\draw    (202.32,97.88) -- (440.63,97.75) ;
\draw [shift={(442.63,97.75)}, rotate = 179.97] [color={rgb, 255:red, 0; green, 0; blue, 0 }  ][line width=0.75]    (10.93,-3.29) .. controls (6.95,-1.4) and (3.31,-0.3) .. (0,0) .. controls (3.31,0.3) and (6.95,1.4) .. (10.93,3.29)   ;

% Text Node
\draw (102,120) node [anchor=north west][inner sep=0.75pt]   [align=left] {\footnotesize mother information (n = 3,798,079)};
% Text Node
\draw (102,171) node [anchor=north west][inner sep=0.75pt]   [align=left] {\footnotesize birth in [1947-01-01, 1976-12-31] (n = 1,131,499)};
% Text Node
\draw (90,290) node [anchor=north west][inner sep=0.75pt]   [align=left] {\footnotesize cases (n = 19,055) };
% Text Node
\draw (260,290) node [anchor=north west][inner sep=0.75pt]   [align=left] {\footnotesize controls (n = 90,833)};
% Text Node
\draw (451,138) node [anchor=north west][inner sep=0.75pt]   [align=left] {\footnotesize Excluded (n = 2,666,580) \\\footnotesize age between 40 and 70};
% Text Node
\draw (102,230) node [anchor=north west][inner sep=0.75pt]   [align=left] {\footnotesize matched subject sample (n = 114,330)};
% Text Node
\draw (453,192) node [anchor=north west][inner sep=0.75pt]   [align=left] {\footnotesize 5-1 matching on \\\footnotesize year of birth};
% Text Node
\draw (102,9) node [anchor=north west][inner sep=0.75pt]   [align=left] {\footnotesize initial sample (n = 6,633,147)};
% Text Node
\draw (451,27) node [anchor=north west][inner sep=0.75pt]   [align=left] {\footnotesize Excluded (n = 117,876) \\\footnotesize birthday after 30/12/2016};
% Text Node
\draw (102,68) node [anchor=north west][inner sep=0.75pt]   [align=left] {\footnotesize birth < 2016-12-30 (n = 6,515,271)};
% Text Node
\draw (451,80) node [anchor=north west][inner sep=0.75pt]   [align=left] {\footnotesize Excluded (n = 2,717,192)\\\footnotesize No mother info};
\end{tikzpicture}
\caption{consort flow chart for the construction of the Nested Case-Control dataset.}
\label{fig:nccd}
\end{figure}
\newpage
\normalsize

\subsection{Descriptive statistics}
Table \ref{tab:covsummaries} reports some descriptive statistics of the clean dataset. In particular, out of $n=1,603,920$ Swedish women, 47,934 (2.99\%) subjects have been diagnosed with invasive breast cancer, 147,914 (9.22\%) are censored due to death, emigration or DCIS before the end of the follow-up period, and 1,408,072 (87.79\%) are alive at the end of the follow-up without experiencing neither breast cancer onset nor any other censoring events, so that we can consider this percentage as our cured fractions. 

The median follow-up is 53.3 years obtained by the inversion of the censoring indicator when computing the estimates of the survival function. A majority of families (45.06\%) consist of two members, followed by three members (33.7\%) and four members (11.37\%). To provide complete information on family structure, we also report in Table \ref{tab:freqsis} the frequencies of sisters, as they play a critical role in our analysis.  

The first row of Table \ref{tab:freqsis} shows the percentage of families having a specific number of sisters for the ``main subjects'', where it can be observed that no main subject has precisely eleven sisters. The second row presents the cumulative percentage of sisters for the main subjects, which is equivalent to the non-missing information on the sister columns. For instance, the first row shows that 36$\%$ of main subjects have precisely one sister (``Freq.'' in the first column), while the 52\% have at least one sister (``Cum. Freq.'' in the second column).
\begin{table}[ht]
    \centering
    \begin{tabular}{l|cccccccccccc}
    \hline 
    N. of sisters &0 &1 &2 &3 &4 &5 &6 &7 &$\ge$8 \\ 
    Freq. &0.48 &0.36 &0.12 &0.03 &0.01 &3.1$\cdot10^{-3}$ &9$\cdot10^{-3}$ &0.0005 \\
    Cum. Freq.  &1 &0.52 &0.16 &0.04 &0.01 &0.005 &0.001 &0.0008 &0.0002 \\ 
    % Number of sisters & &9 &10 &11 &12 \\ 
    % Freq. &1$\cdot10^{-3}$ &3.8$\cdot10^{-4}$ &2.4$\cdot10^{-4}$ &- &0.06$\cdot10^{-4}$ \\
    % Cum. freq. &6.72$\cdot10^{-5}$ &2.92$\cdot10^{-5}$ &5.60$\cdot10^{-6}$ &5.60$\cdot10^{-6}$ \\ 
    \hline
    \end{tabular}
    \caption{frequency table of the number of sisters of the main subjects.}
    \label{tab:freqsis}
% \end{sidewaystable}
\end{table}
% Code from covariates.R in SWEDISH COMPUTER
% code in descriptiveStatistics.R from SWEDISH COMPUTER 

% \begin{center}
% \begin{verbatim}
%
% [1] "Sample size = 1606491"
% [1] "Distribution of family size:"
%      1      2      3      4      5      6      7      8 
% 0.0556 0.4502 0.3370 0.1136 0.0306 0.0087 0.0030 0.0009 
%      9     10     11     12     14 
% 0.0003 0.0001 0.0000 0.0000 0.0000 

% [1] "Fraction of censored main subjects (includes non-cases, if any):"
%      0      1 
% 0.9952 0.0048 

% [1] "Median follow-up (main subjects only):"
% Call: survfit(formula = Surv(registry$x, (1 - registry$delta)) ~ 1)

%            n  events median 0.95LCL 0.95UCL
% [1,] 1606491 1598858   53.4    53.4    53.4
%
% \end{verbatim}
% \end{center}

Returning to the primary dataset, it includes various covariates, including reproductive covariates such as the parity indicator, the number of children for each woman, the age at first child's birth, and the breast cancer family history, that is again defined as having at least a case of invasive breast cancer in the immediate family. All of these variables refer to the most recent follow-up time of each woman. Further details are provided in Table \ref{tab:covsummaries}.
In addition, the data include information on birth date, breast cancer diagnosis (including age at diagnosis), death, and emigration, both overall and only covering the age range of 40 to 70 years.
It is worth noting that the first recorded breast cancer diagnosis occurred in January 1958, with no specific day (we assume the onset date to be the 15th of that month). The final recorded case occurred on December 30th, 2016, providing a 58-year follow-up for analysis. We also consider this date as the end of the follow-up period.
Genetic factors and other details related to breast cancer are not available in the data, limiting the scope of our analysis to familial survival data provided by the women themselves during their visits. No medical tests or blood samples are available.
% \begin{sidewaystable}
\begin{table}
    \centering
    \begin{tabular}{{l|ccccccc}}
    \hline
    &Min. &Median %&Mean 
    &Max. \\ 
Birthday &1853-11-15 &1955-03-15 %&1954-01-15 
&2018-12-15 \\
Diagnosis &1958-01-15 &1994-02-25 %&1992-03-07 
&2016-12-30  \\
Death &1947-05-08 &1991-08-21 %&1991-01-18 
&2018-12-31  \\ 
Emigration &1948-01-01 &1999-09-30 %&1996-09-12 
&2018-12-31 \\
Age at diagnosis Red. &11.00 &50.00 %&50.2 
&69.00 \\
Diagnosis Red. &1961-09-15 &2008-08-13 %&2006-12-27 
&2016-12-30 \\
Death Red. &1948-10-25 &2006-12-26 %&2000-08-10 
&2018-12-31 \\
Emigration Red. &1951-05-01 &1992-08-24 %&1991-02-02 
&2018-12-31 \\ \hline
Observed follow-up &0 &53.2922 %&52.8118 
&69.9576 \\ \hline
BC diagnosis &Yes &No \\
&47,914(0.0299) &1,556,006(0.9701) \\ \hline
EFS &Alive &Dead/emigrated &Diagnosed \\ 
&1,408,072(0.8779) &147,914(0.0922) &47,934(0.0299) \\ \hline
& Yes & No \\
FH at end of FU &105,432(0.0657) &1,498,488(0.9343) \\ 
Parity &98,9422 (0.3831) &614,498 (0.6169)    \\ \hline
&Min. &Median &Mean &Max. \\ 
Number of children &1 &1 &1.5 &10 \\
Age at first child &13.08 &27.25 &27.67 &60.33 \\ \hline
    \end{tabular}
    \caption{summaries of the main variables obtained from the Multi-Generational Swedish Dataset, where ``Red.'' means the birthday range window between 1947-01-01 and 1976-12-31. Notice that the acronym of EFS is ``status at the end of the follow-up'', and ``FU'' is for ``follow-up''.}
    \label{tab:covsummaries}
\end{table}
% code in descriptiveStatistics.R from SWEDISH COMPUTER 
% \end{sidewaystable}

We now construct the survival information, comprising the observed time and the indicator of having observed invasive breast cancer. Once the dataset has been cleaned and merged across families and all necessary subject information is collected, the observed time for an individual is determined based on whether she has experienced breast cancer onset, death, or last emigration out of Sweden. If the individual has not experienced any of these events, her observed time is the day the follow-up ends. 

In Figure \ref{fig:kmall}, we present the Kaplan-Meier estimator of the survival function for all subjects in the dataset, whose family's size ranges from a minimum of one to a maximum of fourteen subjects. 
\begin{figure}
    \centering
    % \includegraphics[width=0.7\linewidth]{plots/Rplot84.pdf}
    \includegraphics[width=\linewidth]{plots/Rplot100.pdf}
    \caption{Kaplan-Meier survival estimate of age at onset (all subjects).}
    \label{fig:kmall}
\end{figure}
% code in sisters.R SWEDEN COMPUTER  

Let us turn to the parameter estimation step.

Notice that in each family, the main subject may or may have not survival information on the mother. The analysis is conducted both on those main subjects that has a recorded mother in the Multi-Generational Dataset and on all main subjects (with or without recorded mother in the dataset). Additionally, a preliminary analysis is conducted on the semiparametric models with o without reproductive covariates such as the parity indicator, the number of children, and the age at first child in categories (with a category for those who have not yet had a child) in Table \ref{tab:14} without giving so different results from the covariate-free scenario. We conduct only this preliminary analysis because it is the easiest and fastest. We plan to extend the study on the covariates also to the other models, and first of all in a simulation study. 
\clearpage
\subsection{Semiparametric models}
In Section \ref{sec5} three semiparametric models have been studied in simulation studies, and then the analysis has moved to the parametric scenario. Not surprisingly, based on the simulation results, the Multivariate Shared Frailty Cox (MSFCox) outperforms all others in terms of predictive accuracy, and also does it in the available dataset, with a concordance index of 0.965, which is significantly higher than the Univariate FH Cox (UFHCox) model (0.5150) and Time-varying version with 0.5036 of concordance (as reported in Table \ref{tab:13}).
When we select only main subjects with the complete information on the mother, the concordance almost reaches the maximum value, up to a 99\% of concordant pairs. This result is outstanding, and the advantage of using the multivariate model instead of only the easier $FH$ model is immediately observed. Indeed, the Constant $FH$ and Time-varying $FH(t)$ models perform poorly, as their prediction concordance is comparable to that of a random flipping of a coin. Equally to the simulation study results, in bold is reported the best result by scenario, say by box, and the best across all scenario is highlighted.  
% The posterior frailty distributions of both the MSFCox and the UFCox models on the dataset without covariates are shown in Figure \ref{fig:postfraildens}. 

% \begin{figure}[ht]
%     \centering
%     \includegraphics[width=0.7\linewidth]{plots/Rplot85.pdf}
%     \caption{Posterior frailty density from the multivariate model (in red) versus the prior (in blue).}
%     \label{fig:postfraildens}
% \end{figure}
% \newline

% The key information obtained from the inference step is the estimated frailty parameter from the multivariate likelihood. The complete analysis output is presented in Table \ref{tab:13}, with standard errors of the estimated coefficients shown in parentheses. The family history variable is included as both an end-of-follow-up indicator and a time-varying covariate, resulting in two versions of the FHCox model. Parameter estimation is performed on the entire dataset, as well as on a subset of subjects with at least maternal information and born between 1946 and 1974. Additionally, the analysis is repeated with the inclusion of covariates such as parity, number of children, and age at first child. Results of the estimated parameter values can be found in Table \ref{tab:14}.

% \begin{table}[ht]
%     \centering
%     \begin{tabular}{l|ccc}
%     \hline
%     Model & Estimated $\widehat{\theta}$ &$\widehat{\beta}_{FH}$ & Concordance \\ \hline
%     MSFCOx &1.327 &- &\textbf{0.965} (se = 0) \\
%     UFCox &200 &- &- \\ 
%     UFHCox &- &1.449 (0.0173) &0.515 (se = 0.001) \\ \hline
%     \end{tabular}
%     \caption{Frailty parameter, family history coefficient and concordance estimated for three models: Multivariate Shared Frailty Cox (MSFCOx), Univariate Frailty Cox (UFCox) and Univariate FH Cox (UFHCox) model.}
%     \label{tab:ouputreal}
% \end{table}

\begin{table}[ht]
    \centering
    \begin{tabular}{l|cccc} \hline
    &Survival information mother &$\widehat\theta$ &$\widehat\beta_F$ &C \\
    Multivariate &yes &1.3624 &- &\hl{\textbf{0.9968}} \\
    Univariate FH &&- &1.5372 &0.5021 \\
    Univariate FH(t) &&- &1.8017 &0.5042 \\ \hline
    Multivariate &no &1.327 &- &\textbf{0.9650} \\
    Univariate FH &&- &1.449 &0.5150 \\ 
    Univariate FH(t) &&- &1.8405 &0.5036 \\ \hline
    \end{tabular}
    \caption{estimated parameters and Harrell's concordance index C (last column) for the three semiparametric models.}
    \label{tab:13}
\end{table}
% Code \texttt{finalEstimation-cox-swemultireg-MVV} in the Swedish VDI.

In Table \ref{tab:14} we report results from including the available reproductive covariates parity and number of children. No significant differences in terms of concordance index is noticed in comparison to the scenario without covariates.
\begin{table}[ht]
    \centering
    \begin{tabular}{l|cccc} 
    \hline
    &Survival information mother &$\widehat\theta$ &$\widehat\beta_F$ &C \\
    Multivariate &yes &2 &- &\textbf{0.9335} \\
    Univariate FH &&- &1.3922 &0.6209 \\
    Univariate FH(t) &&- &2.2254 &0.5350 \\ \hline
    Multivariate &no &2 &- &\hl{\textbf{0.9436}} \\
    Univariate FH &&- &1.3922 &0.5965 \\ 
    Univariate FH(t) &&- &1.8049 &0.5184 \\ \hline
    \end{tabular}
    \caption{estimated parameters and Harrell's concordance index for the three semiparametric models with parity, number of children, age at first child covariates.}
    \label{tab:14}
\end{table}
Also, we can not conclude anything about parameter recovery but the estimate of the frailty parameter $\widehat\theta$ in the second column of Table \ref{tab:13} has a reasonable value. Additionally, the estimated of the family history coefficient $\widehat\beta_F$ has a positive value (around 1.8) meaning that the family history has a positive effect on the breast cancer development, that is indeed coherent with this phenomenon.  
% Code \texttt{finalEstimationCovariates-cox-swemultireg-MVV} in the SWEDISH COMPUTER.
\newpage
% Code in ParContinuousFrailty-multivariate-swemultireg-MVV.R in the SWEDISH COMPUTER for the estimation of parameters and ParContinuousFrailty-multiunifh-postprediction-swemultireg-MVV.R for concordance with adjustement on the parameter values.  
We then use the estimated parameters to perform posterior prediction of familial frailty. Firstly, we apply the posterior prediction to the subjects in the initial dataset. Secondly, we perform an in-sample validation to assess the accuracy of the prediction. This involves defining a training and a validation set by conducting a in-sample validation. 
% Specifically, we remove one family at a time, fit the model on the remaining (n - 1) families, and predict only on the excluded family using the previously estimated parameters.

In other words, the algorithm takes as input the familial survival data of the female family members of a new woman. By providing information on her mother and sisters, the algorithm produces three quantities: the posterior mean frailty, the probability of belonging to the highest-risk group (top 5\%), and an indicator of whether the woman belongs to the highest-risk families compared to the population distribution of the probability of belonging to the highest-risk group, based on a fixed threshold. For example, from the semiparametric multivariate model we obtain that $\widehat{\theta} = 1.327$, and $\widehat{r}_{(1-\alpha)}$ is $0.1512$ for $\alpha = 0.05$. This means that if a woman has a probability of belonging to the highest-risk families over 15.12\%, then her indicator will take value one. We can determine whether the woman should be targeted, e.g., for a more careful treatment path with more intensive screening, surveillance, and other prevention strategies. This prediction can be made quickly and easily using the estimated frailty parameter, the (1-$\alpha$) percentile value, and the Breslow estimates of the cumulative hazard function obtained by fitting the multivariate model. These values can be stored for later use, enabling fast prediction without having to recompute the entire process for new women. This algorithm has been developed based on the model chosen in Section \ref{sec5}.

In Table \ref{tab:17}, a summary of the distribution of the cumulative hazard function is provided, specifically in the semiparametric scenario, and a plot of the cumulative hazard function is in Figure \ref{fig:8}, and its density from the entire population is reported in Figure \ref{fig:9}. Interestingly, we can notice how it can be split into two regions. It is worth noting that in the parametric scenario, the population parametric distribution provides more information and therefore there is no requirement for computing the nonparametric estimates of the cumulative hazard function and the quantile threshold. It is straightforward to extend this procedure to other percentile levels, although this necessitates refitting the model from the beginning, which can be time-consuming, nut we are certain that we can compute it for once and store it for later use. 
\begin{table}[ht]
    \centering
    \begin{tabular}{ccccccc}
    \hline
    Min. &1st Qu. &Median &Mean &3rd Qu. &Max. &NA's \\ 
    0.0000  &0.0000 &0.0324 &0.0752 &0.1389 &0.2583 &360 \\ \hline
    \end{tabular}
    \caption{estimated population Breslow cumulative hazard function summary on a grid of time values.}
    \label{tab:17}
\end{table}
\begin{figure}
    \centering
    \includegraphics[width=\linewidth]{PhD_thesis_template/plots/Rplot107.pdf}
    \caption{estimated Breslow cumulative hazard function.}
    \label{fig:8}
\end{figure}
\newpage
\begin{figure}
    \centering\includegraphics[width=\linewidth]{PhD_thesis_template/plots/Rplot102.pdf}
    \caption{density plot of the cumulative hazard function estimated from the entire population in object.}
    \label{fig:9}
\end{figure}
\newpage
An example is reported below for a woman with a family survival information as follows. Observed ages = (45, 90, 60) in years, and indicators of having observed the onset event = (0, 1, 0). Consider the first subject to be the woman who shows up at the hospital at 45 years of age, with the mother who had breast cancer onset at 90, and a sister who has not experienced the onset yet at age 60. The output from the algorithm is given by the following, which means that the woman does not belong to the highest-risk families: \\
\begin{verbatim}
    Posterior mean frailty = 1.5762
    Posterior median frailty = 2.7801
    Posterior high-risk probability = 0.1264
    Posterior high-risk membership = 0
\end{verbatim}
% Code with name \texttt{postPredictionReal1Family} and in Appendix \ref{appendix:b}. 
The significance of the observed times and breast cancer onset indicators on the probability of belonging to the highest-risk group is thus established. 

\subsection{Focus on the cure-rate}
The rich Swedish dataset also provides us with a precious opportunity to explore the debated issue of whether a cure-rate structure is reasonable in models for breast cancer onset. 

In order to evaluate the goodness of fit of the cure-rate structure for this dataset we now move to the parametric scenario and explore some parametric baseline survival function. Notice that this analysis gives some qualitative support to the hypothesis that a cure-rate exists, thus the frailty quantity here is not involved yet. We examine five distributions, namely Weibull, Gamma, Lognormal, three-parameters Gamma, and three-parameters Lognormal. We explore the multivariate and the univariate setting, fitting both the multivariate and the univariate multiplicative Lehmann frailty cure-rate and non-cure-rate models. Recall that in the univariate model, we only consider the survival information of the main subject per family, while in the multivariate model we consider all of the family survival information.

Results from the analysis are presented in Table \ref{tab:10}, where the models are compared using the Akaike Information Criterion (AIC) that is exactly \[
AIC = -2 n_{\pi} - \ln L(\pi),
\] with $n_\pi$ the dimension of the parameter collection, and $L(\pi)$ the likelihood on the parameters. It should be noted that the models are not nested. We compare the fit across models and between the cure-rate and non-cure-rate survival structure but always within the multivariate and the univariate cases (which also have very different sample sizes). The Multivariate Cure-Rate three-parameters Lognormal model yields the best result, with an AIC value of 1687555, while the regular Lognormal distribution provided the best performance for the Univariate Cure-Rate model with an AIC of 438368.4. 
\begin{table}[ht]
\centering
\begin{tabular}{l|cccc} 
\hline
% &\multicolumn{2}{c}{Multivariate model} &\multicolumn{2}{c}{Univariate model} \\
% \cline{2-5}
% &\multicolumn{2}{c}{AIC} &\multicolumn{2}{c}{AIC} \\
Multivariate &non-cure &cure \\ % &non-cure &cure \\ \hline
Weibull &1705353 &1692822 \\ 
Gamma &1698245 &1688066 \\ 
Lognormal &1693854 &2334626 \\ 
Gamma 3-parameters &1707589 &1687749 \\
Lognormal 3-parameters &1698257 &\textbf{1687555} \\ \hline
Univariate \\ 
Weibull &440685.6 &439053.4 \\ 
Gamma &439516 &438391.8 \\
Lognormal &438958.8 &\hl{\textbf{438368.4}} \\ 
Gamma 3-parameters &444890.7 &438386.2 \\
Lognormal 3-parameters &439460.9 &438369 \\ \hline
\end{tabular}
\caption{AIC comparison among different distributions, where in bold are the best results column-wise and according to the fitted model.}
\label{tab:10}
\end{table}
% Code in oneGroupGamma.R, oneGroupLogNormal.R, oneGroupWeibullNecessary.R in SWEDISH COMPUTER

All the curves shown in Figures \ref{fig:1.13}, \ref{fig:1.14}, and \ref{fig:1.15} support the hypothesis of a cure-rate model and show good closeness to the observed survival distribution better with the cure-rate estimated survival than the non-cure estimated survival function. Indeed, the cure-rate model seems to fit the data (until the end of the follow-up) better than the non-cure-rate models. From results, the cure-rate models are always preferred to the non-cure-rate models (except for the case of multivariate Lognormal model).

It is interesting to notice that support to the cure-rate structure is mainly given by the graphical analysis of the survival function of the mothers, due to their older ages: in Figure \ref{kmsisters}, the Kaplan-Meier curves by subjects (the main subject, the mother, and from the first sister to the last one) show that the tail of the Kaplan-Meier estimator and of the fitted cure-rate models (Figure \ref{fig:1.13}, \ref{fig:1.14}, \ref{fig:1.15}) is mostly attributable by the mothers (in black). A deepening about the reliability of the cure-rate structure and the heavy tail due to the presence of the oldest mothers is in Appendix \ref{appendix:1.e}. We do not find particular graphical differences in curves among the daughters, as it should be expected since the main subject is randomly sampled among all the sisters and so it is the ordering of the sisters. We do not consider in this graphical analysis the last sisters (the twelfth) because of the very limited number of subjects. We note here that one might extend the models to allow for a (say, polynomial) effect of birth cohort on the survival distribution and make distinction between the mother and the sisters.
\begin{figure}
    \centering
    \includegraphics[width=\linewidth]{PhD_thesis_template/plots/Rplot108.pdf}
    \includegraphics[width=\linewidth]{PhD_thesis_template/plots/Rplot109.pdf}
   \caption{Kaplan-Meier for all subjects, in comparison to the estimated survival curves following the cure-rate (green) and the non-cure survival structure (blue) with Weibull and Gamma baseline survival function, on top and above, respectively.}
    \label{fig:1.13}
\end{figure}
\newline
\begin{figure}
    \centering
    \includegraphics[width=\linewidth]{plots/Rplot111.pdf}
        \includegraphics[width=\linewidth]{plots/Rplot112.pdf}
    \caption{Kaplan-Meier for all subjects, in comparison to the estimated survival curves following the cure-rate (green) and the non-cure survival structure (blue), with Lognormal and three-parameters Lognormal survival function, on top and above, respectively.}
    \label{fig:1.14}
\end{figure}
\newline
\begin{figure}
    \centering
    \includegraphics[width=\linewidth]{plots/Rplot110.pdf}
    \caption{Kaplan-Meier for all subjects, in comparison to the estimated survival curves following the cure-rate (green) and the non-cure survival structure (blue), with a three-parameters Gamma baseline survival function.}
    \label{fig:1.15}
\end{figure}
% Code for these plots is in: KMSubjectsMothers_Ultracentenary.R in the Swedish computer. 
\newline
\begin{figure}[ht]
    \centering
    \includegraphics[width=\linewidth]{plots/Rplot106.pdf}
    \caption{Kaplan-Meier of daughters, included the main subject, and mother (in black).}
    \label{kmsisters}
\end{figure}
% code in sisters.R SWEDEN COMPUTER 
\clearpage

However, on top of the AIC values, we are interested in the accuracy of the models to describe the observed cure-rate structure. 

We report the values of the estimated parameters in Table \ref{tab:1.17} and \ref{tab:1.18}. We run the analysis both on main subjects with reported survival information on the mother and on all main subjects (with the Weibull distribution fixed as the baseline survival distribution). In Table \ref{tab:1.18} we compare different baseline survival distribution on all the main subjects. Notice that we can not assess the model fitting accuracy by comparing the estimated parameters value with true values, because we are with a real dataset, but the estimated cured fraction value has a reasonable value for this specific application in the range, for example for $\widehat p$ in the first column of Table \ref{tab:1.17}, around 87\%-96\%. In Tables also the estimates of the baseline survival distribution parameters are reported, that are $\widehat shape_0$, and $\widehat scale_0$ for the Weibull and Gamma, with the addition of $\widehat\gamma_0$ for the threshold parameter in the three-parameters Gamma; $\widehat\mu_0$, and $\widehat\sigma^2_0$ for the Lognormal, with the addition of $\widehat\gamma_0$ for the threshold parameter in the three-parameters Lognormal. The frailty parameter $\widehat\theta$ and the $FH$ coefficient $\widehat\beta_F$ are reported in the same column for a matter of space, because also notice that each model estimates one or the other. 
\begin{table}[ht]
    \centering
    \begin{tabular}{l|cccccc} \hline
    &Survival info mother &$\widehat p$ &$\widehat{shape}_0$ &$\widehat{scale}_0$ &$\widehat\theta$/$\widehat\beta_F$ &C \\
    Multivariate &yes &0.8710  &6.2645 &73.2018  &4.9053 &\textbf{0.9592} \\
    Univariate &&0.9594 &0.1246 &0.1900 &1.2150 &0.3952 \\
    Univariate FH &&0.9635 &0.0926 &0.0394 &1.2174 &- \\ \hline
    Multivariate &no &0.8703  &6.1822 &72.9776  &4.3569 &\hl{\textbf{0.9663}} \\
    Univariate &&0.9750 &0.1176 &0.1193 &1.2614 &0.3963 \\  
    Univariate FH &&0.9658 &0.0965 &0.0362 &1.1982 &- \\ \hline
    \end{tabular}
    \caption{estimated parameters and concordance index for the three parametric models.}
    \label{tab:1.17}
\end{table}
% Code in ParContinuousFrailty-multidistribution-multivariate-swemultireg-MVV.R, ParContinuousFrailty-multidistribution-univariate-swemultireg-MVV.R, ParContinuousFrailty-multidistribution-univariate-fh-swemultireg-MVV.R in the SWEDISH COMPUTER for the estimation of parameters and ParContinuousFrailty-multiunifh-postprediction-swemultireg-MVV.R for concordance with adjustement on the parameter values.  
\begin{table}
    \centering
    \begin{tabular}{l|cccccc} \hline
    Multivariate &$\widehat p_0$ &$\widehat\mu_0 / \widehat{shape}_0$ &$\widehat\sigma^2_0 / \widehat{scale}_0$ &$\widehat\gamma_0$ &$\widehat\theta$/$\widehat\beta_F$ &C \\
    Weibull &0.8703  &6.1822 &72.9776 &- &4.3569 &\textbf{0.9663} \\ 
    Gamma &0.8552 &18.6981  &3.8617 &- &5.4790 &\textbf{0.9663} \\ 
    Lognormal &0.8419 &4.2911 &0.2604 &- &5.5265 &\textbf{0.9392} \\ 
    Gamma 3-pars &0.8529 &18.6685 &3.8172 &0.1546 &5.8116 &\hl{\textbf{0.9679}} \\
    Lognormal 3-pars &0.8408 &4.2746 &0.2590 &2.7922 &5.9126 &\textbf{0.9663} \\ \hline
    Univariate \\
    Weibull &0.9750 &0.1176 &0.1193 &- &1.2614 &0.3963 \\ 
    Gamma &0.8626 &5.9127 &11.5267 &- &-5.8713 &0.3963 \\ 
    Lognormal &0.6607 &4.5317 &0.3784 &- &1.4119 &0.5105 \\ 
    Gamma 3-pars &0.3250 &9.5974 &11.0534 &3.1358 &-1.4014 &0.4461 \\
    Lognormal 3-pars &0.2527 &4.8125 &0.4431 &11.5269 &2.0759 &0.49 \\ \hline
    Univariate FH \\
    Weibull &0.9635 &0.0926 &0.0394 &- &1.2174 &- \\ 
    Gamma &0.9590 &22.0724  &2.5169 &- &-1.1910 &- \\ 
    Lognormal &0.9774 &2.6356 &0.9717 &- &-3.0954 &- \\ 
    Gamma 3-pars &0.7076 &11.8656 &6.9557 &0.5153 &0.7802 &- \\
    Lognormal 3-pars &0.5237 &4.6639 &0.4117 &7.9716 &2.0892 &- \\ \hline
    \end{tabular}
    \caption{estimated parameters through the maximization of the multivariate, univariate likelihood or univariate likelihood with inclusion of family history as covariate, with baseline survival distribution varying.}
    \label{tab:1.18}
\end{table}
% codice su server meb: ParContinuousFrailty-multidistribution-multivariate-swemultireg-MVV.R, ParContinuousFrailty-multidistribution-univariate-swemultireg-MVV.R, ParContinuousFrailty-multidistribution-univariate-fh-swemultireg-MVV.R and ParContinuousFrailty-multiunifh-postprediction-swemultireg-MVV.R for concordance with adjustement on the parameter values. 
%  code for concordance multi-uni-unifh is in prova-concordance-2.R SWEDEN 

The conclusion coincides to the one in the semiparametric scenario. The multivariate model outperforms the other two univariate models compared in terms of prediction accuracy through the Harrell's concordance index C, reported in the last column of Tables \ref{tab:1.17}, and \ref{tab:1.18}. It has indeed around the 96\% of concordant pairs. Also, in comparison to the semiparametric scenario we can say that the estimated frailty parameter has a higher value in the multivariate parametric scenario bringing to assess that the frailty distribution here has a higher variance and thus this brings to an easier distinction among low-risk and higher-risk families, than the semiparametric scenario. 

Now we can predict the posterior family-specific frailty as shown in Section \ref{sec:1.6}. We compute the mean, median and mode of the posterior frailty distribution for each family. Later, we compute the posterior high-risk probability and the posterior high-risk membership indicator, fixed a frailty mean threshold. Here, too, we can also compute the whole estimated family-specific survival distribution, conditionally on the family survival information. For each woman we can compute thus, as in the semiparametric scenario, all the interesting quantities to assess whether to address her to prevention strategies targeted for highest-risk families. 
\newpage

\section{Discussion}\label{sec:1.7}
This study aims to contribute to the study of risk prediction models for breast cancer to better understand the nature and modeling of family-specific risk of onset. We consider the cure-rate structure as a realistic approach for the Swedish Multi-Generational Breast Cancer registry, where around the 85$\%$ of subjects have not experienced yet breast cancer onset within the observed follow-up. We thus extend the traditional proportional hazards assumption in this Lehmann family formulation to cure-rate models. We explore parametric (Gamma shared frailty) models in this cure-rate setting, as well as semiparametric models that do not assume the cure-rate structure. 

Although family information is crucial for risk prediction models for breast cancer, using only a summary of this information may not have enough predictive power, as it proved with the often used family history indicator (baseline covariate, or time-varying covariate). Our simulation-based comparison shows that a full multivariate framework induces much better performance in terms of accuracy in risk prediction, when only involving family membership without additional subject-specific covariates. A full assessment of the added value of the full multivariate model will clearly emerge when additional analyses can be conducted on other dataset. Including family-specific covariates can enhance precision in targeting and improve accuracy in identifying the frailty parameter, as well as classifying families into risk groups. Therefore, incorporating family-specific covariates is an intriguing extension worth exploring.

We believe that this work stands out among the others known in the literature as it improves several aspects of breast cancer risk prediction models. We report a deep comparison between our model and the BOADICEA model, that is one of the most powerful in this context. 

The BOADICEA model is based on a multiplicative hazard function that depends on a genetic frailty component. This approach consists of inferring a genetic latent quantity, the polygenic risk score (PRS), for predicting cancer risk based on the family history of the disease and other risk factors. The PRS is a weighted combination of single nucleotide polymorphisms (SNPs), which are genetic mutations, commonly spread into the population, that singularly give a small contribution to increase the risk of breast cancer, but that can be dangerous when combined all together.

The BOADICEA model is then univariate and based on the subject-specific hazard function \[ \lambda(t\mid r) = \lambda_0(t)exp(\beta(r)), \] as described in detail in several publications (\cite{antoniou2002comprehensive}, \cite{antoniou2004boadicea}, \cite{antoniou2008boadicea}, and \cite{thomas2004statistical}). It evaluates the likelihood function through a complex segregation analysis provided by the Mendel software \cite{lee2019boadicea}. 

In contrast, our proposed frailty model differs from the BOADICEA model (or other possible similar models) in several ways. While BOADICEA uses a subject-specific hazard within a univariate framework and infers a genetic latent quantity, our model works in a fully multivariate framework and we infer a generic risk latent quantity, i.e. the subject-specific polygenic score. We incorporate survival information in a family-specific hazard, while the BOADICEA hazard function is subject-specific and does not account explicitly for the family structure through the estimated genetic frailty component. Our model wins in the simplicity that it has involving only familial breast cancer information and no other factors such as genetic, as BOADICEA does. Nevertheless, it is not difficult to extend our work to the use of additional  covariates. The simplicity is also extended to the use of a full likelihood or partial likelihood maximization algorithm for parameter estimation, while BOADICEA relies on a complex segregation analysis to predict the PRS (specifically its variance).

Last but not least, at contrary to the BOADICEA model, we introduce the explicit cure-rate structure on the survival function, that is crucial when especially dealing with breast cancer risk prediction models.

Talking about possible extensions of our work, one could be the use of alternative frailty distributions, such as the Lognormal \cite{duchateau2008frailty}. Or, one could address the problem within the Bayesian framework \cite{karamoozian2021bayesian}. One could also compare the prediction accuracy between our multivariate -- semiparametric and parametric -- model and the BOADICEA model, but it would be necessary the access to the same complete dataset \cite{lee2017differences}.

% The method with the multivariate model is able to identify higher-risk families to address them to targeted screening surveillance and prevention paths.

% The family information has been proved to be crucial to gain more information on the risk of having breast cancer. The Univariate model has been proved to be not useful for this topic. The model is not identifiable so far, without the inclusion of valid familiar covariates. Of course this work can be readily extended to the use of covariates and compare again the multivariate model with the univariate to assess whether the available covariates could be useful for knowing the familiar risk to breast caner. The issue with this dataset is that we have not precise familiar information, but we can build some as the number of people in the family or the number of sons. 

% Unfortunately all the families are from Sweden and they are registered as Swedish citizen so we can not distinguish among countries. Also other information are equivalent over all the families and indeed not useful. Without covariates, the multivariate model not only performs better than the univariate model because it uses the survival family information but also it is more powerful than the family history indicator in terms of prediction accuracy. The family history indicator uses as well the family information but in a simplistic way, indeed we can appreciate, through this study, the huge advantage in using the extended family information instead of just a summary. 

% The adjusted mean results to be correlated to the real frailty in the simulation studies, but the summary error index related to the regular mean is always lower according the every family size maximum/minimum. This means that we prefer to use an index that is closer to the real frailty, and indeed the mean is used in the application. 

% There is shrinkage towards one when the expected number of events is very low. There a large amount of shrinkage in the cure-rate model when the families have few members within. It is interesting to study the behaviour of the posterior estimator of the frailty close to zero because the shrinkage is heavily towards the mean in our case. 

% compile first the main.tex, then run Bibtex on Chapter1.bib, Chapter2.bib and Chapter3.bib, finally compile main.tex again twice
\clearpage
\renewcommand{\bibname}{References}
\bibliographystyle{apalike}
\bibliography{biblio}
